Opens in a new window Opens an external website Opens an external website in a new window
Close this dialog
This website utilises technologies such as cookies to enable essential site functionality, as well as for analytics, personalisation, and targeted advertising. You may change your settings at any time or accept the default settings. You may close this banner to continue with only essential cookies.  Privacy Policy
Manage Preferences  Accept All  Reject Non-Essential 
Close Cookie Preferences
  * Skip to Article Content
  * Skip to Article Information


[](https://nph.onlinelibrary.wiley.com/)
[](https://nph.onlinelibrary.wiley.com/)
Search withinThis Journal New Phytologist Journals Wiley Online Library
  * Search term
 Advanced Search Citation Search
  * Search term
 Advanced Search Citation Search
  * Search term
 Advanced Search Citation Search


Login / Register
  *  Individual login 
  *  Institutional login 
  *  REGISTER 


[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370#main1 "menu drawer")
  * Journals
    * New Phytologist 
    * Plants, People, Planet 


  * newphytologist.org
     
     


#  Journal list menu 
  * Journal
  * __Articles


  * Actions


PLANTS, PEOPLE, PLANET
Volume 7, Issue 2 pp. 448-466[](https://nph.onlinelibrary.wiley.com/journal/25722611 "PLANTS, PEOPLE, PLANET homepage")
RESEARCH ARTICLE
Open Access
[](http://creativecommons.org/licenses/by/4.0/)
# The seed transcriptome of _Rafflesia_ reveals horizontal gene transfer and convergent evolution: Implications for conserving the world's largest flower
Jeanmaire Molina, 
Corresponding Author
Jeanmaire Molina
  * jmolina2@pace.edu
  * jeanmaire.molina@liu.edu


  * orcid.org/0000-0001-5250-2684


Biology Department, Pace University, New York, New York, USA
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
**Correspondence**
Jeanmaire Molina, Biology Department, Pace University, New York, NY, USA.
Email: jmolina2@pace.edujeanmaire.molina@liu.edu
Search for more papers by this author
Adhityo Wicaksono, 
Adhityo Wicaksono
  * orcid.org/0000-0003-3298-3577


Division of Biotechnology, Genbinesia Foundation, Gresik, Indonesia
Study Group of Bioinformatics, Synthetic Biology Indonesia, Sumbawa, Indonesia
Search for more papers by this author
Todd P. Michael, 
Todd P. Michael
The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
Cepheid, Sunnyvale, California, USA
Search for more papers by this author
Su-Hwan Kwak, 
Su-Hwan Kwak
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Ronniel D. Pedales, 
Ronniel D. Pedales
  * orcid.org/0000-0002-5335-6313


Institute of Biology, University of the Philippines Diliman, Quezon City, Philippines
Search for more papers by this author
Zoé Joly-Lopez, 
Zoé Joly-Lopez
  * orcid.org/0000-0002-7926-322X


Département de Chimie, Université du Québec à Montréal, Montréal, Québec, Canada
Search for more papers by this author
Semar Petrus, 
Semar Petrus
Cepheid, Sunnyvale, California, USA
Search for more papers by this author
Allen Mamerto, 
Allen Mamerto
The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
Search for more papers by this author
Brian Tomek, 
Brian Tomek
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Sumaya Ahmed, 
Sumaya Ahmed
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Venkatasivasankar Maddu, 
Venkatasivasankar Maddu
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Kristina Yakubova, 
Kristina Yakubova
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Danilo Tandang, 
Danilo Tandang
  * orcid.org/0000-0003-2708-661X


Botany and National Herbarium Division, National Museum of the Philippines, Manila, Philippines
Search for more papers by this author
Joseph W. Morin, 
Joseph W. Morin
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
So-Yon Park, 
So-Yon Park
  * orcid.org/0000-0002-2126-3382


Division of Plant Science and Biotechnology, University of Missouri, Columbia, Missouri, USA
Search for more papers by this author
Hyun-Oh Lee, 
Hyun-Oh Lee
Division of Plant Science and Biotechnology, University of Missouri, Columbia, Missouri, USA
Search for more papers by this author
William McLaughlin, 
William McLaughlin
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Kyle Wallick, 
Kyle Wallick
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
James Adams, 
James Adams
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Ari Novy, 
Ari Novy
  * orcid.org/0000-0002-0873-9948


The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
San Diego Botanic Garden, Encinitas, California, USA
Search for more papers by this author
Susan Pell, 
Susan Pell
  * orcid.org/0000-0002-7214-3225


U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Michael D. Purugganan, 
Michael D. Purugganan
  * orcid.org/0000-0002-9197-4112


Center for Genomics & Systems Biology, New York University, New York, USA
Search for more papers by this author
Jeanmaire Molina, 
Corresponding Author
Jeanmaire Molina
  * jmolina2@pace.edu
  * jeanmaire.molina@liu.edu


  * orcid.org/0000-0001-5250-2684


Biology Department, Pace University, New York, New York, USA
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
**Correspondence**
Jeanmaire Molina, Biology Department, Pace University, New York, NY, USA.
Email: jmolina2@pace.edujeanmaire.molina@liu.edu
Search for more papers by this author
Adhityo Wicaksono, 
Adhityo Wicaksono
  * orcid.org/0000-0003-3298-3577


Division of Biotechnology, Genbinesia Foundation, Gresik, Indonesia
Study Group of Bioinformatics, Synthetic Biology Indonesia, Sumbawa, Indonesia
Search for more papers by this author
Todd P. Michael, 
Todd P. Michael
The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
Cepheid, Sunnyvale, California, USA
Search for more papers by this author
Su-Hwan Kwak, 
Su-Hwan Kwak
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Ronniel D. Pedales, 
Ronniel D. Pedales
  * orcid.org/0000-0002-5335-6313


Institute of Biology, University of the Philippines Diliman, Quezon City, Philippines
Search for more papers by this author
Zoé Joly-Lopez, 
Zoé Joly-Lopez
  * orcid.org/0000-0002-7926-322X


Département de Chimie, Université du Québec à Montréal, Montréal, Québec, Canada
Search for more papers by this author
Semar Petrus, 
Semar Petrus
Cepheid, Sunnyvale, California, USA
Search for more papers by this author
Allen Mamerto, 
Allen Mamerto
The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
Search for more papers by this author
Brian Tomek, 
Brian Tomek
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Sumaya Ahmed, 
Sumaya Ahmed
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Venkatasivasankar Maddu, 
Venkatasivasankar Maddu
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Kristina Yakubova, 
Kristina Yakubova
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
Danilo Tandang, 
Danilo Tandang
  * orcid.org/0000-0003-2708-661X


Botany and National Herbarium Division, National Museum of the Philippines, Manila, Philippines
Search for more papers by this author
Joseph W. Morin, 
Joseph W. Morin
Department of Life Sciences, Long Island University, Brooklyn, New York, USA
Search for more papers by this author
So-Yon Park, 
So-Yon Park
  * orcid.org/0000-0002-2126-3382


Division of Plant Science and Biotechnology, University of Missouri, Columbia, Missouri, USA
Search for more papers by this author
Hyun-Oh Lee, 
Hyun-Oh Lee
Division of Plant Science and Biotechnology, University of Missouri, Columbia, Missouri, USA
Search for more papers by this author
William McLaughlin, 
William McLaughlin
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Kyle Wallick, 
Kyle Wallick
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
James Adams, 
James Adams
U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Ari Novy, 
Ari Novy
  * orcid.org/0000-0002-0873-9948


The Plant Molecular and Cellular Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California, USA
San Diego Botanic Garden, Encinitas, California, USA
Search for more papers by this author
Susan Pell, 
Susan Pell
  * orcid.org/0000-0002-7214-3225


U.S. Botanic Garden, Washington, District of Columbia, USA
Search for more papers by this author
Michael D. Purugganan, 
Michael D. Purugganan
  * orcid.org/0000-0002-9197-4112


Center for Genomics & Systems Biology, New York University, New York, USA
Search for more papers by this author
First published: 13 April 2023
<https://doi.org/10.1002/ppp3.10370>
Citations: 11
[About](javascript:void\(0\))
  * ## Figures
  * ## References
  * ## Related
  * ## Information
  * PDF


[Sections](javascript:void\(0\))
PDF
Tools
  * Request permission
  * Export citation
  * Add to favorites
  * Track citation


ShareShare
Give access
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
Close modal
Share full-text access
Please review our Terms and Conditions of Use and check box below to share full-text version of article.
I have read and accept the Wiley Online Library Terms and Conditions of Use
* * *
Shareable Link
Use the link below to share a full-text version of this article with your friends and colleagues. Learn more.
Copy URL
Share a link
Share on
  *   *   *   *   *   *   * 

## Societal Impact Statement
en
_Rafflesia_ is a genus of parasitic plants with the largest flowers in the world, unique to the threatened forest habitats of tropical Asia. Here, we report on genes that are active (the transcriptome) in _Rafflesia_ seeds as part of a larger effort to understand _Rafflesia_. _Rafflesia_ has never been grown successfully outside of its native range. Consequently, seed banking is not yet possible, precluding a critical management strategy for conservation. The study of _Rafflesia_ seed biology is a critical step to improve its cultivation, which will educate the public about unique species and the importance of conserving their habitats.
###  Summary
  * _Rafflesia_ is of great interest as one of the only two plants known to have completely lost its chloroplast genome. _Rafflesia_ is a holoparasite and an endophyte that lives inside the tissues of its host, a tropical grape vine (_Tetrastigma_), emerging only to bloom—with the largest flower of any plant. Here, we report the first _Rafflesia_ seed transcriptome and compare it with those of other plants to deepen our understanding of its extraordinary life history.
  * We assembled a transcriptome from RNA extracted from seeds of the Philippine endemic _Rafflesia speciosa_ and compared this with those of other plants, including _Arabidopsis_ , parasitic plants _Striga_ and _Cuscuta_ , and the mycoheterotrophic orchid _Anoectochilus_.
  * Genetic and metabolic seed pathways in _Rafflesia_ were generally similar to the other plant species. However, there were some notable exceptions. We found evidence of horizontal transfer of a gene potentially involved in circumventing host defenses. Moreover, we identified a possible convergence among parasitic plants because _Rafflesia_ , _Striga_ , and _Cuscuta_ shared important similarities. We were unable to find evidence of genes involved in mycorrhizal symbiosis, suggesting that mycoheterotrophy is unlikely to play a role in _Rafflesia_ parasitism.
  * To date, ex situ propagation of _Rafflesia_ by seed has been mostly unsuccessful. Our research is a bold step forward in understanding the fundamentals of _Rafflesia_ seed biology that will inform the continued propagation and seed-banking efforts concerning this recalcitrant plant. We discuss our findings in the broader context of the conservation of a genus in peril.


## Abstract
tl
Ang _Rafflesia_ ay isang uri ng parasitikong halaman na may pinakadambuhalang bulaklak sa buong mundo. Ito ay matatagpuan lamang sa mga nanganganib na tropikal na kagubatan ng Asya. Hindi pa napalalago ang _Rafflesia_ sa labas ng Asya. Dahil dito, hindi pa posible ang seed banking na mahalaga sa konserbasyon ng _Rafflesia_. Pinag-aralan namin dito ang genetics ng buto o liso ng _Rafflesia_ upang maunawaan ang mga prosesong nagaganap sa loob nito. Ang pag-aaral na ito ay isang kritikal na hakbang sa pagpapalago ng _Rafflesia_ upang ito ay mapreserba at maipalaganap ang malasakit ng publiko sa mga namumukod tanging halaman at lupalop na kanilang pinanggagalingan.
## 1 INTRODUCTION
Since first described in the 18th century from Java, Indonesia, the enormous blooms (~1 m) of _Rafflesia_ that smell like rotting meat have mesmerized scientists and lay naturalists alike. Unlike most plants, _Rafflesia_ (Rafflesiaceae) is a parasite bereft of stems, roots, and leaves. It lives as an endophyte, growing within the tissues of its host vine, the genus _Tetrastigma_ (Vitaceae), and only emerges from its host's exterior to flower (Davis et al., 2007). Genomic studies have demonstrated an association between _Rafflesia_ and _Tetrastigma_ predating the origin of _Tetrastigma_ itself, suggesting other taxa in the Vitaceae may have even served as hosts in the past (Cai et al., 2021). An association dating back to the Cretaceous may have afforded _Rafflesia_ , and the closely-related _Sapria_ , to become the only plants we know of to completely discard their plastid genomes (Cai et al., 2021; Molina et al., 2014) while retaining their plastid compartments for amino acid and lipid synthesis (Ng et al., 2018). The loss of the plastid genome is so far unique in Rafflesiaceae, that a science news article even asked, “When is a plant no longer a plant? (Pennisi, 2014).” However, its massive flowers attest to its bona fide “plant status.”
There are about 40+ _Rafflesia_ spp. endemic to the dwindling forests of tropical Asia with several species considered critically endangered (Barcelona et al., 2009; Pelser et al., 2019). Dubbed panda of the plant world, _Rafflesia_ is a charismatic icon of plant conservation, but unlike the panda, efforts to conserve and cultivate _Rafflesia_ out of its natural habitat have been limited and challenging (Molina et al., 2017; Wicaksono et al., 2016, 2021). This is perhaps a direct consequence of the little that we know of its biology, especially of its seeds. Small mammals, like shrews previously observed to gnaw on the hard covering of the fruits, are believed to facilitate seed dispersal (Bänziger, 2004; Nais, 2001), though ants have also been seen transporting the seeds (Pelser et al., 2013, 2016). How these seeds germinate and infect the host is unknown (Wicaksono et al., 2021), but after the seed germinates in the host, it seems that the _Rafflesia_ endophyte can persist in the vegetative stage for years (M. Gabin, personal communication, July 28, 2016; Bascos et al., 2021).
Molina et al. (2017) inoculated _Rafflesia_ seeds in uninfected _Tetrastigma_ host species growing in the US Botanic Garden, following techniques by Marius Gabin, a _Rafflesia_ grower in Malaysia, but the emergence of the bud has yet to be seen. Ex situ propagation attempts from _Rafflesia_ seed have been unsuccessful except for one—a blooming flower of _Rafflesia arnoldii_ was reported in 2022 in Bogor Botanic Garden, Indonesia, presumably a result of a seed inoculation experiment several years prior (S. Mursidawati and D. Latifah, personal communication, January 5, 2023). However, grafting _Rafflesia-_ infected _Tetrastigma_ to an uninfected rootstock seems to be the most viable method as implemented in Bogor Botanic Garden, producing multiple blooms for public display (Mursidawati et al., 2015; Thorogood et al., 2021), and exemplifies the importance of these emblematic species in promoting plant conservation awareness. To date, only one species, _Rafflesia patma_ , has been propagated ex situ (Mursidawati et al., 2015). It is estimated that 76% of parasitic plant diversity is missing in botanic gardens (Thorogood et al., 2021), including rare holoparasites like _Rafflesia_ , translating to lost opportunity to raise awareness and appreciation for these evolutionary marvels and their threatened habitats.
In vitro experiments to induce _Rafflesia_ seed germination using various plant growth regulators such as gibberellins, ethylene, brassinolide, and cytokinin have also been unsuccessful (Molina et al., 2017). Even the synthetic strigol, GR24, known to stimulate seed germination in plant parasites of Orobanchaceae, did not induce germination. Transcriptome sequencing of germinating seeds of the Orobanchaceae parasite, _Phelipanche aegyptiaca_ exposed to GR24 revealed numerous differentially expressed genes related to DNA, RNA, protein repair and biosynthesis, carbohydrate, and energy metabolism. Upregulation of abscisic acid (ABA), gibberellin (GA), and ethylene-associated genes were also observed, with seed germination related to the increasing concentration of GA and ethylene and concomitant reduction in ABA (Yao et al., 2016). In fact, these plant hormone interactions during seed germination are not unique to plant parasites and generally apply to many plants, including the model species _Arabidopsis_ (Carrera-Castano et al., 2020).
Ranjan et al. (2014) profiled the transcriptomes of different developmental stages of the holoparasite _Cuscuta pentagona_ (Convolvulaceae), and in the pre-haustoria seedling stage, they found an enrichment of genes associated with responses to stress, biotic, and endogenous stimuli, as well as transporter, kinase, and catalytic activities. The haustorium is the specialized feeding structure in many plant parasites that regulates transfer of nutrients from host to parasite (Teixeira-Costa, 2021; Yang et al., 2015). Yang et al. (2015) studied the genetic basis of haustorial development in plant parasites of Orobanchaceae, namely, _Phelipanche_ , _Tryphysaria_ , and _Striga_ , characterizing the shared upregulated genes as “parasitism genes,” having enhanced expression in the haustoria. These included genes for proteolysis such as serine carboxypeptidase and aspartyl protease, cell wall modification genes (e.g., pectate lyase), and peroxidase genes. They discovered that orthologs of these haustorial genes were expressed significantly in root and pollen tissues in nonparasitic model plant species suggesting that haustorium genes evolved from these nonparasitic but similarly intrusive plant structures.
Unlike the seeds of Orobanchaceae which are responsive to host-derived strigolactones (Wang et al., 2017), it is yet unknown what triggers germination and development of the _Rafflesia_ seed. In this study, we seek a better understanding of _Rafflesia_ seed biology by comparing a newly generated transcriptome of _Rafflesia speciosa_ seeds to published seed transcriptomes of _Arabidopsis_ (Dorone et al., 2020), _Striga_ (Yang et al., 2015), _Cuscuta_ (Ranjan et al., 2014), and _Anoectochilus_ (Liu et al., 2015). The orchid _Anoectochilus_ is initially mycoheterotrophic during seed germination, reliant on mycorrhizal fungi due to lack of nutritive reserves in the minute seeds until leafy structures begin to photosynthesize (Liu et al., 2015). _Rafflesia_ seeds are comparatively tiny (0.7–0.9 mm; Figure 1) with scant oily endosperm enveloping the tiny undifferentiated embryo (Bouman & Meijer, 1994). JM (unpublished) has characterized diverse endophytic microbiota in _R. speciosa_ seeds with plant-growth promoting properties in the literature (Khalaf & Raizada, 2016). It is possible that _Rafflesia_ may be similarly dependent on microbial symbionts during germination, similar to _Anoectochilus_ and other mycorrhizal-dependent orchids. To our knowledge, this study presents the first seed transcriptome of _Rafflesia_. Here, we present the seed genetics of this unique holoparasite and describe its similarities and differences to other plants, including other parasites and a mycoheterotrophic orchid. We discuss how insights from the _Rafflesia_ seed transcriptome can benefit future ex situ propagation and conservation efforts of the world's largest flower.
 
**FIGURE 1**
Open in figure viewerPowerPoint
Plants in this study and their corresponding seed anatomy. (a) _Rafflesia speciosa_ (photo by J. Molina, scale bar = 10 cm), seed anatomy modified from Wicaksono et al. (2021) to reveal the internal tissue of the embryo based on Ng et al. (2018); (b) _Cuscuta pentagona_ (photo by Mason Brock, scale bar = 1 cm), seed anatomy inspired by Olszewski et al. (2020); (c) _Anoectochilus roxburghii_ (image from Ye et al., 2017, _J Ethnopharm_ 209: 184–202, with permission, scale = 5 cm), seed anatomy inspired by Li et al. (2019); (d) _Striga hermonthica_ (photo from USDA ARS, public domain via Wikimedia Commons, scale = 5 cm), seed anatomy inspired by Seed Identification Guide (2018), www.idseed.org; and (e) _Arabidopsis thaliana_ (photo by SH Kwak, scale = 5 cm), seed anatomy inspired by Haughn and Chaudhury (2005). The images of _C. pentagona_ and _S. hermonthica_ were taken by Mason Brock and the United States Department of Agriculture: Agricultural Research Service, respectively and are available in Wikimedia public domain with no copyright. The image of _Anoectochilus roxburghii_ is added by permission from Elsevier.
## 2 MATERIALS AND METHODS
### 2.1 RNA extraction and library preparation
Seeds (0.5 g) from one fruit of _R. speciosa_ were collected in August 2016 at Miagao, Iloilo, Philippines with permission from the Philippine government (Gratuitous Permit 242). The seeds were scraped from the dehisced fruit and air dried for transport to the United States with the appropriate transport and export permits. The dried seeds were ground in liquid nitrogen with total RNA extracted using Qiagen RNeasy Plant Mini Kit (Cat. No. 74904) following manufacturer's protocol resulting in 6.24 ng/μL of extracted RNA. Eluted RNA was treated with Ambion TURBO DNA-free kit (Thermo Fisher Scientific, Cat. No. AM1907) and then quantified by Qubit RNA BR assay. Ribo-Zero rRNA removal procedure was performed according to the manufacturer's instructions. rRNA-depleted RNA was cleaned up using the Agencourt RNAClean XP Kit (Beckman Coulter, Cat. No. A63987). ScriptSeq Complete Seed/Root kit (Illumina, Cat. No. BSR1206) was used for library prep following the protocol for “Low Input” RNA. The RNA-seq library was purified using the AMPure XP system. Final library quantification and quality control were performed using the Agilent Tape Station. Libraries were sequenced using Illumina protocols for 2 × 100-bp reads on an Illumina HiSeq 2500 instrument at the New York University GeneCore facility.
### 2.2 _Rafflesia_ seed transcriptome assembly
Illumina reads were filtered and trimmed for quality using Trimmomatic (v0.27). The resulting high-quality reads were assembled with Trinity (v2.8.5) into contigs. For comparison, raw data files for seed transcriptomes of _Striga hermonthica_ (StHe0GB1, imbibed seed Yang et al., [2015]), _C. pentagona_ (SRR966236, seed condition whether dry/imbibed not stated Ranjan et al., [2014]), _Anoectochilus roxburghii_ (SRR2962491, dry seeds Liu et al., [2015]), and _Arabidopsis thaliana_ (GSM5100805, dry seeds Dorone et al., [2020]) were downloaded and independently do novo assembled using Velvet 1.2.10 in Geneious Prime (Biomatters Ltd.) with default parameters.
The fasta files of the assembled contigs of all five taxa were each imported into OmicsBox (Biobam Bioinformatics, Valencia, Spain), blasted against Magnoliopsida nonredundant protein sequences using blastx-fast algorithm with e-val 1.0e−100. For _Rafflesia_ , additional blasting against fungi was performed, and blast results were compared with those obtained for Magnoliopsida. Transcripts that had higher similarity values for fungal genes versus plant genes were eliminated in further analyses as these transcripts may be from endophytic fungi within _Rafflesia_ seeds. After filtering, Salmon (v1.6.0) was used to map the reads to the transcriptome followed by transcript quantification for quality control. Gene ontology (GO) mapping (Götz et al., 2008) using the latest database version was performed on all protein blast hits and then annotated with Blast2GO. Statistics related to Blast hits (top hit species), annotation (direct GO count biological process, molecular function, and cell component), and enzyme code mapping were generated for each transcriptome.
## 3 RESULTS
The read set (158,907,700) had an overall mapping rate of 92.1% and determined to be suitable for downstream analysis. Ninety-four percent (135,913,373) of our mapped reads were associated with highly expressed transcripts (TPM > 1000). Of the ~90K assembled _Rafflesia_ seed contigs, over 7K had blast hits using Blastx-fast algorithm (1.0e−100) in OmicsBox against nonredundant protein sequences of Magnoliopsida. Ninety-one percent (91%) of these had GO annotations (Table 1). Comparable statistics for other plant taxa are also given in Table 1. Table 2 provides the proportion of contigs in the _Rafflesia_ seed transcriptome. The top 10 blast hits for _Rafflesia_ (Figure 2) included Malpighiales taxa (_Hevea_ , _Manihot_ , _Jatropha_ , _Ricinus_ , and _Populus_ , total 49% transcripts) to which _Rafflesia_ belongs, though 2% transcripts were most similar to _Vitis_ of Vitaceae, the host family. We do not think this is contamination from the host because the _Rafflesia_ seeds were enclosed in the partially dehisced fruit and the seeds were not touching the host root.
TABLE 1. Descriptive statistics of the de novo assembled transcripts for taxa in this study including _Rafflesia_.
Plant taxon and reference | De novo assembled contigs | Number of BLAST hits | Percentage with GO annotations from number of BLAST hits  
---|---|---|---  
_Arabidopsis_ (dry, GSM5100805; Dorone et al., 2020) | 20,236 | 10,995 | 93.4  
_Anoechtochilus_ (dry, SRR2962491; Liu et al., 2015) | 63,274 | 5367 | 76.7  
_Cuscuta_ (unknown seed condition, SRR966236; Ranjan et al., 2014) | 29,037 | 2120 | 83.2  
_Rafflesia_ (air-dried, this study) | 89,950 | 7025 | 91.2  
_Striga_ (imbibed, StHe0GB1; Yang et al., 2015) | 13,872 | 5584 | 98.5  
  * _Note_ : Published raw data sets for _Anoechtochilus_ , _Arabidopsis_ , _Cuscuta_ , and _Striga_ were downloaded and de novo assembled producing contigs (>300 bp) that were blasted and annotated.
  * Abbreviation: GO, gene ontology.


TABLE 2. Contig statistics for the _Rafflesia_ seed transcriptome.
Length range (bp) | Count | Percentage  
---|---|---  
300–600 | 121 | 1.7  
601–1000 | 1978 | 28.2  
1001–2000 | 4011 | 57.1  
2001–3000 | 681 | 9.7  
3001–4000 | 167 | 2.4  
4000+ | 67 | 0.9  
Total | 7025 | 100  
  * _Note_ : The proportion of contigs with a certain length, from those with BLAST hits, is given.
  * Abbreviation: bp, base pairs.


 
**FIGURE 2**
Open in figure viewerPowerPoint
Top blast hits of the _Rafflesia_ seed transcripts. The top hits included Malpighiales taxa (_Hevea_ , _Manihot_ , _Jatropha_ , _Ricinus_ , _Populus_ , total 49% transcripts) to which _Rafflesia_ belongs, though 2% transcripts were most similar to _Vitis_ of Vitaceae, the host family.
Because of the differences in sequence coverage across datasets resulting in discrepant counts for assembled contigs, we compared the proportions for GO counts for biological process (BP), molecular function, and cellular component (Figure 3) for each taxon (#GO for a specific category / total #GO for the taxon, e.g., 228 _Rafflesia_ transcripts for protein phosphorylation divided by all _Rafflesia_ transcripts annotated under BP). The most enriched GO functions for BP in the _Rafflesia_ seed were “protein phosphorylation,” “proteolysis,” and “phosphorylation,” though still proportionally lower compared with other taxa (Figure 3a). As for molecular function GO, _Rafflesia_ seed transcripts were enriched for “ATP binding,” “metal ion binding,” and “ATP hydrolysis activity,” like other taxa (Figure 3b). However, for cellular component (Figure 3c), _Rafflesia_ seed values were below average for “nucleus” but higher for “cytoplasm” which may be related to the higher activity of cytoplasmic-associated Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways such as “protein processing in endoplasmic reticulum (ER),” “glycolysis/gluconeogenesis,” “pyruvate metabolism,” “oxidative phosphorylation,” “peroxisome,” and “fatty acid degradation.” “mRNA surveillance pathway and beta-alanine metabolism” were also increased (Figure 4). The _Rafflesia_ seed transcriptome was proportionally higher for fatty acid metabolism, compared with the seed transcriptomes of the other taxa studied here, based on the heatmap of homologous oil metabolism genes (Figure 5) involved in seed germination (Miray et al., 2021).
 
**FIGURE 3**
Open in figure viewerPowerPoint
(a) Top biological processes in _Rafflesia_ seed compared with _Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_. The top three enriched processes in _Rafflesia_ were “protein phosphorylation,” “proteolysis,” and “phosphorylation,” though still relatively lower compared with other taxa. Proportion of _Rafflesia_ seed transcripts was the lowest for “protein ubiquitination,” “regulation of transcription, DNA-templated,” and “DNA repair,” though comparatively higher for “proton transmembrane transport” compared with the other taxa. (b) Top molecular function processes in _Rafflesia_ seed compared with other taxa. _Rafflesia_ seed transcripts were enriched for “ATP binding,” “metal ion binding,” and “ATP hydrolysis activity,” like other taxa. _Rafflesia_ was relatively lower for “DNA binding” and “protein serine/threonine kinase activity” yet higher in “nucleotide binding” and “magnesium ion binding” compared with the other taxa. (c) Top cellular component structures in _Rafflesia_ seed compared with other taxa. _Rafflesia_ transcripts were enriched for “integral component of membrane,” “cytoplasm,” and “nucleus.”
 
**FIGURE 4**
Open in figure viewerPowerPoint
Top Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways in _Rafflesia_ seed transcriptome compared with _Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_. _Rafflesia_ seed transcripts were relatively higher for the following pathways (indicated with “*” and discussed): mRNA surveillance pathway, glycolysis/gluconeogenesis, pyruvate metabolism, oxidative phosphorylation, proteasome, fatty acid degradation, and beta-alanine metabolism compared with other taxa.
 
**FIGURE 5**
Open in figure viewerPowerPoint
Heatmap of oil metabolism genes involved in seed germination. _Rafflesia_ seed transcriptome was proportionally higher for fatty acid metabolism (cf. Figure 4) compared with the seed transcriptomes of the other taxa studied (_Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_).
Sixty-seven percent (67%) of _Rafflesia_ transcripts were annotated as enzymes. Similar proportional trends for various enzyme classes were apparent across taxa (Figure 6) with transferases as the most abundant enzyme class in all seed transcriptomes examined (>35%), and “transferring phosphorus-containing groups” is the top function in all taxa (results not shown). Hydrolases composed about 30% of the enzymes present in all seed transcriptomes. Oxidoreductases made up over 10% of seed transcripts.
 
**FIGURE 6**
Open in figure viewerPowerPoint
Enzyme classes in _Rafflesia_ seed compared with _Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_. Sixty-seven percent (4732/7025) of _Rafflesia_ transcripts were annotated as enzymes.
Figure 7, which was adapted from Carrera-Castano et al. (2020), depicts the interplay of various phytohormones in germination and dormancy. We discuss below potential interaction of phytohormonal response proteins and germination-dormancy molecular regulators detected in the _Rafflesia_ seed, as shown in Figure 8, in regulating its development. The _Rafflesia_ seed had transcriptionally active genes for GA, brassinosteroid, and salicylic acid, which were all active in _Striga_ and _Arabidopsis_ , the latter having additional hormone biosynthesis genes present. Interestingly, the _Rafflesia_ seed had higher proportion of GA synthesis genes compared with other taxa (Figure 9).
 
**FIGURE 7**
Open in figure viewerPowerPoint
Regulatory network of phytohormones involved in germination and dormancy. Figure was adapted from Carrera-Castano et al. (2020). The interaction of auxins (AUX), cytokinins (CKs), gibberellic acids/gibberellins (GA), abscisic acids (ABA), ethylene (ETH), brassinosteroid (BR), and karrikin (KR) controls seed development. Colored outlines are used to distinguish each hormone.
 
**FIGURE 8**
Open in figure viewerPowerPoint
Heatmap of phytohormonal response proteins and germination-dormancy molecular regulators, showing the proportion of seed transcripts for each species. Levels of auxin response factor were similarly higher across taxa investigated (_Anoectochilus_ , _Cuscuta_ , _Rafflesia_ , _Striga_ , and _Arabidopsis_).
 
**FIGURE 9**
Open in figure viewerPowerPoint
Gene ontology count heatmap of phytohormone biosynthesis genes in _Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_. The _Rafflesia_ seed had transcriptionally active genes for gibberellic acids/gibberellins, brassinosteroid, and salicylic acid, with higher proportion of gibberellic acids/gibberellins synthesis genes compared with the other taxa.
The transcripts of regulatory factors such as TRANSPARENT TESTA GLABRA1 (TTG1), HOMEODOMAIN GLABRA2, and GLABRA2-EXPRESSION MODULATOR, as well as other metabolic genes that are involved in phenylpropanoid synthesis pathway, were identified in the _Rafflesia_ seed transcriptome (Table 3), including cinnamoyl-CoA reductase—a key enzyme in lignin biosynthesis (Table 3). Our results also corroborate previous studies on the absence of a plastome (Cai et al., 2021; Molina et al., 2014). We did not detect any plastid-encoded mRNA in _Rafflesia_ seeds. However, _psbA_ , which encodes for the D1 protein of Photosystem II, was present in the other four taxa studied including the plant parasites _Striga_ and _Cuscuta_ as well as the mycorrhizae-dependent seed _Anoectochilus_ (Figure 10). Over 2% of the transcripts had _Vitis_ as the top hit (vs. a member of Malpighiales) and were presumably horizontally transferred (HT; Table 4). Many transcripts are involved in “DNA integration” including transposon integration perhaps mediated by “nucleic acid phosphodiester bond hydrolysis,” another enriched GO. Gene trees for SERK, RHM1, thiC, and glutamyl-tRNA (Gln) amidotransferase subunit A demonstrate putative horizontal transfers from the host family (Figure 11).
TABLE 3. Transcripts that are involved in phenylpropanoid synthesis in _Rafflesia_ seed transcriptome.
Identified transcripts | Gene name |  _Rafflesia_ contig | Expect value | Species of identified homolog  
---|---|---|---|---  
Regulatory factors |  |  |  |   
TRANSPARENT TESTA GLABRA1 | _TTG1_ | 41185 | 9.73E−116 | _Manihot esculenta_  
GLABRA2-EXPRESSION MODULATOR | _GEM_ | 86527 | 2.30E−119 | _M. esculenta_  
GLABRA2-EXPRESSION MODULATOR | _GEM_ | 86534 | 6.34E−119 | _M. esculenta_  
GLABRA2-EXPRESSION MODULATOR | _GEM_ | 35727 | 1.31E−116 | _Ricinus communis_  
HOMEODOMAIN GLABRA2 | _HDG2_ | 19125 | 1.03E−159 | _Populus alba_  
Phenylpropanoid pathway enzymes |  |  |  |   
Phenylalanine ammonia-lyase | _PAL_ | 73687 | 0.00E+00 | _R. communis_  
Phenylalanine ammonia-lyase | _PAL_ | 73688 | 0.00E+00 | _R. communis_  
4-Coumarate-CoA ligase | _4CL3_ | 86494 | 0.00E+00 | _Jatropha curcas_  
Chalcone synthase | _CHS_ | 53706 | 0.00E+00 | _Actinidia chinensis_  
Flavonoid 3′-monooxygenase | _F3'H, TT7_ | 86571 | 0.00E+00 | _P. alba_  
Dihydroflavonol 4-reductase | _DFR, TT3_ | 30307 | 5.14E−135 | _M. esculenta_  
Leucoanthocyanidin reductase | _LAR_ | 60680 | 2.96E−167 | _Quercus suber_  
Cinnamoyl-CoA reductase | _CCR_ | 76321 | 5.01E−108 | _Populus euphratica_  
Cinnamoyl-CoA reductase | _CCR_ | 46001 | 3.55E−116 | _Salix brachista_  
Anthocyanidin 3-O-glucosyltransferase | _UGT_ | 10343 | 1.94E−168 | _Herrania umbratica_  
Anthocyanidin 3-O-glucosyltransferase | _UGT_ | 10348 | 2.49E−170 | _H. umbratica_  
Anthocyanin 5-aromatic acyltransferase | _AACT1, ACT_ | 37598 | 1.21E−134 | _R. communis_  
Anthocyanin 5-aromatic acyltransferase | _AACT1, ACT_ | 37594 | 8.33E−137 | _R. communis_  
Flavonoid transporter |  |  |  |   
TRANSPARENT TESTA9 | _TT9_ | 74015 | 7.14E−114 | _R. communis_  
TRANSPARENT TESTA9 | _TT9_ | 74018 | 3.24E−126 | _R. communis_  
 
**FIGURE 10**
Open in figure viewerPowerPoint
Heatmap of plastid-related genes across the five taxa (_Rafflesia_ , _Anoectochilus_ , _Cuscuta_ , _Striga_ , and _Arabidopsis_)_._ Genes for light-dependent reaction (a), C3 carbon fixation (b), C4-CAM carbon fixation (c), starch synthesis process (d), and carotene synthesis (e). All enzymes are noted with their respective Kyoto Encyclopedia of Genes and Genomes enzyme IDs (except for the light dependent because some are composed of multiple proteins).
TABLE 4. _Rafflesia_ seed transcripts by GO biological process with top blastx hits to _Vitis_ (vs. members of Malpighiales) which may indicate putatively horizontally transferred transcripts.
GO biological process | Transcript count  
---|---  
DNA integration | 36  
Proteolysis | 21  
RNA-templated DNA biosynthetic process | 13  
Nucleic acid phosphodiester bond hydrolysis | 10  
Glutathione metabolic process | 7  
Protein phosphorylation | 5  
Peptide catabolic process | 4  
Others | 96  
  * Abbreviation: GO, gene ontology.


 
**FIGURE 11**
Open in figure viewerPowerPoint
Gene trees of _Rafflesia_ seed transcripts showing evidence of horizontal gene transfer from the host family (Vitaceae). (a) SERK; (b) RHM1 (rhamnose biosynthesis 1); (c) thiC (phosphomethylpyrimidine synthase gene); and (d) glutamyl-tRNA (Gln) amidotransferase subunit A. BlastX (NCBI) was performed on the _Rafflesia_ seed transcripts against the taxa shown in the phylogenies including _Vitis_ (representative of the host family, Vitaceae) and Euphorbiaceae (the closest photosynthetic relative of _Rafflesia_ , represented by _Manihot_ and _Hevea_). Multiple sequence alignment was performed using MAFFT v 7.490, followed by phylogeny reconstruction using PhyML 3.3.20180621 (with LG substitution model and SH-branch like supports), as implemented in Geneious Prime 2022.2.1 (Biomatters, Ltd).
## 4 DISCUSSION
### 4.1 Seed comparison
To better understand the genetics of the _Rafflesia_ seed, we generated its transcriptome and compared it with de novo assembled seed transcriptomes from the plant parasites, _Cuscuta_ and _Striga_ ; the mycorrhizal orchid, _Anoectochilus_ ; and the model photosynthetic plant, _Arabidopsis_ (Table 1). The seeds of the different taxa compared here are all tiny with an embryo and scant endosperm surrounded by a seed coat (Figure 1). _Rafflesia_ seed has a distinctive bilobed shape with tiny crevices all over its woody seed coat. Within the seed, the oily endosperm is one layered and envelopes the tiny embryo, with 14–22 cells and undifferentiated cotyledons (Baskin & Baskin, 2022). The mycorrhizal-dependent _Anoectochilus roxburgii_ seed is twice the length (~2 mm) of the _Rafflesia_ seed. Proteins and lipids are stored within the embryo proper. There is some lignin in the seed coat (Li et al., 2019). _Striga_ seed is even more miniscule, at 0.2–0.35 mm in length, with an endosperm that can only support the embryo for a few days (Runo & Kuria, 2018). _Cuscuta_ seed size (length) ranges from 0.7 to 3 mm with an embryo surrounded by hard endosperm but no cotyledons (Olszewski et al., 2020). Both _Cuscuta_ and _Striga_ can germinate in soil until they exhaust their nutritive reserves, at which point, they need to have colonized their hosts. The nonparasitic _Arabidopsis_ also has a very small seed, ~0.2 mm, composed entirely of embryo filled with lipid and protein and enclosed with one layer of endosperm (Haughn & Chaudhury, 2005).
We were able to assemble >7K contigs from the _Rafflesia_ seed transcriptome, with 4011 (57.1%) contigs 1001–2000 bp in length (Table 2). Forty-nine percent (49%) of the assembled _Rafflesia_ seed transcripts were most similar to Malpighiales taxa to which _Rafflesia_ belongs, as expected (Figure 2). At least 2% transcripts were most similar to _Vitis_ of the grape family, Vitaceae, the host family, which may be a consequence of the rampant horizontal gene transfer between host and parasite given their intimate connection (Cai et al., 2021; Molina et al., 2014; Xi et al., 2012, 2013).
### 4.2 GOs and KEGG pathways
We compared the assembled seed transcriptome of _Rafflesia_ to those of other taxa including _Arabidopsis_ as well as other parasitic plants to determine potential overlap in seed genetic pathways. When the _Rafflesia_ seeds were collected, they were damp from rain and then air-dried for transport. This may have begun the process of water uptake in the seeds. Generally, to break dormancy, a seed has to imbibe water initiating a series of metabolic changes including energy metabolism, repair of DNA, degradation of dormancy-associated mRNAs, and restoration of cellular integrity including mitochondrial repair to begin cell respiration (Nelson et al., 2017). As the seed breaks dormancy, energy metabolism is increased which explains the increased cytoplasm-associated “glycolysis/gluconeogenesis” activity (Figure 3) and the high levels for the KEGG pathway “fatty acid degradation” that occurs in the “peroxisome” (Figures 4 and 5). Oxygen consumption also increases due to increased cell respiration, which may explain why “pyruvate metabolism” and “oxidative phosphorylation” were high in the _Rafflesia_ seed. It is conceivable that the transported fatty acids are converted to sugars (e.g., sucrose, trehalose, xylose, fructose, and glucose) in the _Rafflesia_ seed to fuel the growing embryo (Song & Zhu, 2019). Transcripts associated with the synthesis of these sugars were observed in the _Rafflesia_ seed transcriptome (Data S1). There were also multiple transcripts for pyruvate dehydrogenase (PDHA; Data S1). PDHA converts glucose to acetyl-coA for the tricarboxylic acid cycle (Bao et al., 2017). PDHA expression was found to be significantly upregulated in conditioned and strigol-stimulated _Phelipanche_ seeds of Orobanchaceae (Bao et al., 2017). Metabolism of the nonproteinogenic amino acid beta-alanine was also relatively increased in the _Rafflesia_ seed which may be due to its precursor role in the synthesis of acetyl Co-A (Parthasarathy et al., 2019). It seems that even in the absence of its host, high levels of carbohydrate and lipid metabolism do occur in the _Rafflesia_ seed following imbibition.
Synthesis of proteins involved in cell signaling using the seed-stored mRNAs is also increased (Nelson et al., 2017). The KEGG pathway “mRNA surveillance,” a quality control mechanism that degrades abnormal mRNAs (Kanehisa et al., 2016), was higher in the _Rafflesia_ seed (Figure 4). mRNAs accumulate in seeds during development and are called stored mRNAs/long-lived mRNAs retaining their function as long as the seed is viable and serve as templates for protein synthesis during the germination process. Long-lived mRNAs occur in all plant seeds, even conserved between the dicot _Arabidopsis_ and the monocot barley suggesting functional importance in their role to kickstart protein synthesis in early germination (Zhao et al., 2019). Degradation of certain stored seed mRNAs has been shown to promote seed germination (Nelson et al., 2017), resulting in a more “germination-friendly” transcriptome (Carrera-Castano et al., 2020), which may explain the increased “mRNA surveillance” pathway observed in the _Rafflesia_ seed.
More than two thirds of _Rafflesia_ seed transcripts were annotated as enzymes. Similar proportional trends for various enzyme classes were apparent across taxa (Figure 6), with transferases, hydrolases, and oxidoreductases together making up at least 75% the enzymes. These enzyme classes were also the most abundant types in the _Cuscuta_ whole plant transcriptome (Ranjan et al., 2014). The abundance of hydrolase genes in _Rafflesia_ seed may be explained by the role of hydrolases in cell wall loosening and seed reserve mobilization during seed germination (Cao et al., 2021). Oxidoreductases catalyze redox reactions that produce reactive oxygen species (ROS), and in parasitic plants, development of the haustoria begins with ROS from the parasite converting host cell wall phenols to benzoquinones stimulating haustorium formation (Cui et al., 2018). It is suspected that oxidoreductase-mediated production of ROS may also be involved in _Rafflesia_ seed germination and host infection.
### 4.3 Phytohormone signaling during seed development
In general, two hormones, GA and ABA, play antagonistic roles in seed development, with GA promoting seed germination whereas ABA maintaining seed dormancy. The ratio of GA/ABA is also regulated by other hormones such as AUX, KR, ETH, and BR (Figure 7). At e-val < 0.001, homologs of DELLA and DOG1 (delay of germination), which increase ABA sensitivity, were not observed in the _Rafflesia_ seed transcriptome, though ABI homologs were recovered, along with WRKY (Figure 8; Data S1). WRKY is a transcription factor induced by ABA and may function as a negative growth regulator. Levels of auxin response factors were similarly high across all taxa (Figure 8), as expected, given that this gene family is greatly involved in regulating the development of seeds. Additionally, transcripts for ETR1, which acts as a negative regulator of response to ethylene (The Uniprot Consortium, 2021), a germination stimulant by DOG1 repression, were also detected. Detection of ABI, WRKY, and ETR1 transcripts may indicate that the _Rafflesia_ seeds sampled, though transcriptionally active in the KEGG pathways described above, have yet to actually germinate sensu stricto, that is, when the embryo radicle protrudes from the seed (Carrera-Castano et al., 2020), which in _Rafflesia_ conceivably happens in the presence of its host.
Though transcripts for _ent_ -kaurene synthetase A and _ent-_ kaurenoic acid oxidase, which are vital in GA synthesis, were recovered in the _Rafflesia_ seed (Figure 9), such enzymes are also involved in production of secondary metabolites like terpenoids, which _Rafflesia_ is known to produce (Ng et al., 2018). Homologs of genes for synthesis of carotenoids were not detected at all in the developmental stage of _Rafflesia_ seed sampled, though relevant transcripts were detected in the other seed transcriptomes studied here (Data S1). Chemical inhibition of carotenoid synthesis using fluridone in _Orobanche_ and _Phelipanche_ (Orobanchaceae) has been found to promote their seed germination (Bao et al., 2017) which may suggest that fluridone may not work in inducing _Rafflesia_ germination because carotenoid synthesis genes were not detected. The strigolactone responsive gene D14 (Wang et al., 2017) was also not found in _Rafflesia_ seed which may explain the lack of morphological development in the _Rafflesia_ seed when exposed to the synthetic strigol GR24 (Molina et al., 2017), though KAI2, which is important in perception of karrikins—fire-related compounds that can stimulate seed germination (Wang et al., 2017), was observed. In addition, the F-box protein MAX2, another key component in karrikin (and strigolactone) signaling, was also recovered (Wang et al., 2017). However, in previous germination experiments of _R. speciosa_ seeds, karrikin did not seem to stimulate germination—no protrusion was seen developing on the seeds (Molina et al., 2017), though it could be possible that tested seeds were agamospermous (Nais, 2001). Thus, hand-pollination has been recommended to produce viable seeds (Bänziger, 2004).
### 4.4 Proanthocyanidin synthesis in _Rafflesia_ seed coat
In mature seeds, proanthocyanidins, produced in the phenylpropanoid pathway, accumulate in the innermost layer of the seed coat, oxidizing to a brown color and are known to protect the seed from environmental stresses and improve the longevity of seeds (Debeaujon et al., 2003). Transcripts involved in this pathway were detected in the _Rafflesia_ seed including TTG1. In _A. thaliana_ , TTG1 activates the transcription of downstream transcription factors such as GL2 and HOMEODOMAIN GLABRA2, which then regulate the expression of metabolic genes involved in phenylpropanoid synthesis (Li et al., 2020). It is also possible for the _Rafflesia_ TTG1 to function in trichome development or other cell fate determination processes because TTG1 is also involved in trichome development and root epidermal pattern formation in _A. thaliana_ (Wang et al., 2010). In the unrelated holoparasite, _Balanophora abbreviata_ (Balanophoraceae), trichomes originating from endosperm cells, facilitated host attachment (Arekal & Shivamurthy, 1976). Whether the TTG1 gene is involved in the development of a similar root-like structure in the _Rafflesia_ seed as it germinates waits to be seen. Other metabolic genes involved in phenylpropanoid synthesis pathway, such as cinnamoyl-CoA reductase—a key enzyme in lignin biosynthesis (Kawasaki et al., 2006), were also identified in the _Rafflesia_ seed transcriptome (Table 3).
The presence of the key transcriptional regulator, TTG1, anthocyanidin, and lignin metabolic genes in the _Rafflesia_ seed transcriptome suggests that proanthocyanidins accumulate via phenylpropanoid synthesis pathway in the lignified seed coat of _Rafflesia_ , which explains the brown color of _Rafflesia_ seeds. _Rafflesia_ 's lignified seed coat conceivably protects the seed during its long dormancy (Wicaksono et al., 2021) until a suitable host is found. Proanthocyanidins have also been demonstrated to inhibit seed germination in _Arabidopsis_ (Jia et al., 2012). As mentioned above, though the _Rafflesia_ seeds were transcriptionally active in energy metabolism, detection of the proanthocyanidin-related transcripts may suggest that when the seeds were sampled, germination was still inhibited, pending host stimulation.
### 4.5 Plastid processes
Dry seeds of _Arabidopsis_ already possess a substantial portion of their plastid-encoded mRNA as stored/long-lived mRNA with de novo mRNA synthesis commencing during imbibition (Demarsy et al., 2012). However, none of the genes in the KEGG light-dependent reactions of photosynthesis (Kanehisa et al., 2016) were present in the _Rafflesia_ seed, yet _psbA_ was present in the other four taxa studied including _Striga_ and _Cuscuta_ as well as the mycorrhizae-dependent seed _Anoectochilus_ suggesting the importance of the presence of this gene even in dry seeds (Figure 10). _psbA_ encodes for the D1 protein of Photosystem II that splits water in the presence of light energy and converts this to ATP to fuel subsequent carbon fixation. It seems that even in the plant parasites, _Striga_ (Tuquet et al., 1990) and _Cuscuta_ (Dinelli et al., 1993), light-mediated reactions occur to some extent until they are established in their hosts, and this is especially true in _Cuscuta_ , which is initially photosynthetic. The endophytic nature of _Rafflesia_ may explain _psbA_ loss.
Transcripts for _rpl32_ (35223–35225; Data S1), a ribosomal plastid protein typically encoded in the chloroplast genome, were also observed in the _Rafflesia_ seed transcriptome and, if shown to be indeed localized in the plastome, would contradict the purported unprecedented loss of a plastome in plants (Cai et al., 2021; Molina et al., 2014). The _Rafflesia_ seed _rpl32_ copy was also phylogenetically allied with Euphorbiaceae (results not shown), thus a native copy and not HT. However, it is possible that this gene has moved from the plastome and is now nucleus encoded, as was discovered in _Rafflesia's_ distantly related Malpighiales photosynthetic relative, _Populus_ (Ueda et al., 2007).
The _Rafflesia_ seed transcriptome had multiple transcripts present in all three carbon fixation pathways comparable with the seed transcriptomes of the other taxa examined (Figure 10). Despite the absence of the light-dependent reactions of photosynthesis, carbon fixation does occur in the _Rafflesia_ seed. However, ribulose-1,5-bisphosphate carboxylase-oxygenase gene (EC 4.1.1.39) was not detected in the _Rafflesia_ seed transcriptome, unlike the other four taxa examined. However, _rbcL_ subunit binding proteins alpha and beta, also known as chaperonin 60α and 60β, respectively, were detected in _Rafflesia_ seed and had intact open reading frames, suggesting they are functional. They also strongly phylogenetically cluster with Euphorbiaceae taxa (results not shown), thus are not HT. Both chaperone proteins are nuclear encoded and essential in _rbcL_ folding and binding the subunits of ribulose-1,5-bisphosphate carboxylase-oxygenase, but in _Rafflesia_ , the absence of _rbcL_ may indicate other functions for the chaperone proteins perhaps related to embryogenesis. In _Arabidopsis_ , alpha mutation arrested embryo development (AT2G28000.1, TAIR, 2022), whereas beta mutants, on the other hand, show accelerated cell death to heat shock stress (AT1G55490.1, TAIR, 2022).
### 4.6 Horizontal transfer
Over 2% of the transcripts had _Vitis_ as the top hit (vs. a member of Malpighiales) and presumably HT. Table 4 shows that many transcripts are involved in “DNA integration” including transposon integration perhaps mediated by “nucleic acid phosphodiester bond hydrolysis,” another enriched GO. Multiple _Rafflesia_ seed HT transcripts were annotated as “transposon polyproteins” (Data S1), and an abundance of these was also observed among _Sapria_ HT orthogroups (Cai et al., 2021). “RNA-dependent DNA biosynthetic process” was enriched due to the presence of multiple retrotransposon transcripts (Data S1). Horizontal gene transfer is disproportionately increased in parasitic plants with these host-derived genes being co-opted by the parasite in haustorial function, playing a role in its adaptive evolution (Yang et al., 2016).
Interestingly, in _Rafflesia_ seed, among putatively HT genes, one notable transcript (60334) was annotated as SERK1 and phylogenetically allied to _Vitis_ (Figure 11). SERK family members interact with R (resistance) proteins to activate the hypersensitive immune response (Su et al., 2020). The rice homolog OsSERK1 has been shown to be involved in defense response against the rice blast fungus (Hu et al., 2005). Cowpea _Vigna unguiculata_ possesses RSG-301, which interacts with SERK upon infection by _Striga_. However, resistant _Striga_ strains subvert RSG-301 by secreting the effector Suppressor of Host Resistance 4z that has homology to a region of SERK. This inhibits VuPOB1 necessary for the hypersensitive response. Thus, Suppressor of Host Resistance 4z in _Striga_ acts as a decoy to circumvent host immune responses (Su et al., 2020). Could it be that _Rafflesia_ seed 60334, which is 96% similar to _Vitis_ SERK1, is acting similarly to enable _Rafflesia_ to undermine _Tetrastigma_ host defenses?
Another _Rafflesia_ seed transcript 29691 annotated as RHM1 (rhamnose biosynthesis 1), which converts UDP-D-glucose to UDP-L-rhamnose (The Uniprot Consortium, 2021), was also phylogenetically allied to _Vitis_ (Figure 11), again suggesting HT. We find this interesting because rhamnose is a precursor of primary cell wall components, and it is tempting to speculate that this could be a mechanism of _Rafflesia_ to mimic host cell wall components to avoid host immune response and/or allow _Rafflesia_ to colonize host tissue, as rhamnose synthase genes were demonstrated to be crucial for host attachment and penetration and fungal pathogenicity of _Verticillium dahliae_ (Santhanam et al., 2017).
In Cai et al. (2021), phosphomethylpyrimidine synthase gene, thiC, involved in biosynthesis of pyrimidine was shown to be HT. Analyzing this gene in _Rafflesia_ phylogenetically also showed support for this (transcript 10612; Figure 11), as well as _Rafflesia_ seed transcript 27956 annotated as “glutamyl-tRNA (Gln) amidotransferase subunit A” (Figure 11) which was also shown to be HT in _Rafflesia_ in Xi et al. (2012). Cai et al. (2021) suggested that host-derived HT genes in Rafflesiaceae may substitute for its genes lost during parasitic evolution.
### 4.7 Convergent evolution
Among flowering plants, parasitism has independently evolved at least 12 times, with 1% of all angiosperms parasitic (Nickrent, 2020). There seem to be conserved molecular mechanisms underlying this heterotrophic life history. Across the 12 lineages, parasitic plants similarly secrete cell-wall degrading enzymes and proteolytic enzymes when attached to host plant tissues. Such enzymes have been detected in the haustoria of Orobanchaceae parasites and even in the unrelated stem parasite, _Cuscuta reflexa_ (Convolvulaceae; Ranjan et al., 2014). Yang et al. (2015) termed these “core parasitism genes” in Orobanchaceae, including genes for cell-wall modifying enzymes (cellulase, etc.), pectinolytic enzymes (pectin methylesterase and pectate lyase), and for phytohormone synthesis—genes for auxin, gibberellin, brassinesteroid, ethylene, and strigolactone biosynthesis. These genes were also upregulated in _Cuscuta_ (Ranjan et al., 2014). Here, in the _Rafflesia_ seed transcriptome, we found annotated terms enriched for host-degrading enzymes similar to the “core parasitism genes” of Orobanchaceae (Yang et al., 2015) including cellulases, pectate lyases, and pectin methylesterase, hinting at an evolutionary convergence among disparate lineages of parasitic plants. The homolog of cuscutain gene, a cysteine protease upregulated in the haustoria of the stem parasite _C. reflexa_ important for host infection (Bleischwitz et al., 2010), was also detected in the root parasite _P. aegyptiaca_ (Rehker et al., 2012) and in the _Rafflesia_ seed transcriptome in this study (Data S1; _Rafflesia_ contig 4146). Whether this transcript is involved in _Tetrastigma_ host infection remains to be seen.
Laccase enzyme transcripts were also recovered in both _Striga_ and _Rafflesia_ seed transcriptomes (Data S1). Interestingly, laccases are enzymes that can degrade host lignin, producing haustorium-inducing factors that promoted haustorium formation in _Striga_ and the confamilial _Phtheirospermum_ (Cui et al., 2018). It will be interesting to know if laccase genes in _Rafflesia_ function similarly. Metabolic profiling of _Tetrastigma loheri_ , host of _Rafflesia lagascae_ (Molina et al., 2021), showed the presence of sinapaldehyde, sinapyl alcohol, syringic acid, and acetosyringone, which are molecules that induced haustoria in both _S. hermonthica_ and _Phtheirospermum japonicum_ (Cui et al., 2018). Testing these compounds in germination experiments of _Rafflesia_ seeds may be worthwhile in the future.
During the conditioning of seeds of the parasitic plant _Phelipanche_ , the gene gibberellin 20-oxidase (GA20OX1) was upregulated resulting in increased GA3 levels. However, in the _Rafflesia_ seed, this GA20OX1 was not detected, though present in the seed transcriptome of _Striga_ , which was obtained from imbibed seeds (Yang et al., 2015). Gibberellin 2-oxidase (GA2OX2), which in _Arabidopsis_ deactivates GA and promotes seed dormancy, was detected in the _Rafflesia_ seed instead. Transcripts for methionine adenosyltransferase, aminocyclopropane-carboxylate synthase, and aminocyclopropanecarboxylate oxidase, which are three key enzymes of ethylene biosynthesis, were also detected in _Rafflesia_ seed, and these corresponding genes were upregulated significantly in _Phelipanche_ after GR24/strigol treatment. Taken together, the _Rafflesia_ seed sampled here may have been imbibed, kickstarting the transcription of the early seed processes detected so far but still need to undergo host stimulation. The increased fatty acid breakdown seen in the _Rafflesia_ seed transcriptome may be associated with lipid mobilization and increased gluconeogenesis in the _Rafflesia_ seed to supply sugars to the embryo, as it awaits host stimulation.
In the previously published _Cuscuta_ seed transcriptome (Ranjan et al., 2014, Supplement 13), transcripts involved in “fructose metabolic process,” “mannose metabolic process,” “glucose 6-phosphate,” “glucokinase activity,” “response to heat,” and “hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides” were enriched and were also detected in _Rafflesia_ seed. Moreover, _Rafflesia_ and _Striga_ were similar proportionally for certain pathways such as “galactose metabolism” and “phenylpropanoid metabolism,” in contrast to _Arabidopsis_. In Orobanchaceae, planteose, a galactose-derived molecule, was found as a storage carbohydrate required for early-stage germination (Wakabayashi et al., 2015). Perhaps, a similar carbohydrate is required by the _Rafflesia_ seed during germination which may be causing the uptick in “galactose metabolism.” The similar proportion in “phenylpropanoid metabolism” for _Striga_ and _Rafflesia_ seeds may be due to the synthesis of the lignin components in their seed coats and other proanthocyanidins (Table 3). This woody seed coat perhaps allows _Striga_ seeds to survive for prolonged periods of time (>10 years) in the seed bank prior to germination (Atera et al., 2011), which may also be the case for _Rafflesia_ seeds.
Comparing _Rafflesia_ seed with the mycoheterotrophic seeds of _Anoectochilus_ , we were unable to find in _Rafflesia_ the homolog of the lipid biosynthetic enzyme, RAM2, which promotes arbuscular mycorrhizal symbiosis (Bravo et al., 2017). However, RAM2 was found in mycorrhizal _Anoectochilus_ , as expected, annotated as “glycerol-3-phosphate 2-O-acyltransferase.” The absence of RAM2 in _Rafflesia_ seed may indicate that _Rafflesia_ is not dependent on arbuscular mycorrhiza or that this gene is not yet expressed in the seed developmental stage we sampled.
## 5 CONCLUSIONS AND IMPLICATIONS FOR _RAFFLESIA_ CONSERVATION
In this study, we produced the first _Rafflesia_ seed transcriptome and compared it with seed transcriptomes of a variety of plant species across the phylogenetic tree. We were particularly interested in explaining the difficulty of germinating _Rafflesia_ seeds in the laboratory as well as elucidating metabolic pathways that could be involved in seed development of _Rafflesia_. Overall, metabolic pathways of _Rafflesia_ are similar to the other plant species we compared it to; however, there are notable findings. Our data suggest that following imbibition, the _Rafflesia_ seed is already transcriptionally active, showing increased energy metabolism, preparing the seed for germination upon host stimulation. Our transcriptome results provide further evidence that the plastome is absent in _Rafflesia_ yet rife in multiple transposable elements, many apparently HT from the host, similar to its confamilial relative _Sapria_. Furthermore, we found evidence of genes in the _Rafflesia_ seed putatively HT from the host. _Rafflesia_ could have co-opted these HT genes to potentially inhibit host immune response during infection.
The _Rafflesia_ seed transcriptome is also enriched for many classes of the “core parasitism genes” found in Orobanchaceae. The lack of carotenoid synthesis genes in _Rafflesia_ seed suggests that fluridone may not induce germination as it does in parasitic Orobanchaceae. Similarly, lack of strigolactone responsive genes may explain the absence of morphological development when _Rafflesia_ seeds were exposed to synthetic strigol. However, detection of karrikin response transcripts in _Rafflesia_ seed suggests that karrikin may stimulate _Rafflesia_ seed germination especially if they were collected from fruits produced by hand-pollination. It is also worthwhile inoculating _Rafflesia_ seeds in sinapaldehyde, sinapyl alcohol, syringic acid, and acetosyringone, which are molecules that induced haustoria in Orobanchaceae parasites and are also present in _Tetrastigma_ hosts. Transcripts for host-degrading laccase enzymes implicated in haustorium formation in Orobanchaceae were also detected in _Rafflesia_ , which may suggest that _Rafflesia_ seed inoculation in _Tetrastigma_ may benefit from laccase application. It is also possible that _Rafflesia_ seeds, like _Striga_ seeds, can survive for prolonged periods, given its thick woody seed coat and accumulation of phenylpropanoids. Additionally, we discovered the homolog of the cuscutain gene, important for host infection by _Cuscuta_ , in the _Rafflesia_ seed. The present study thus provides strong evidence for a theory of generalized convergent evolution among disparate lineages of parasitic plants. We were unable to find homologs of genes involved in mycorrhizal symbiosis in orchids, suggesting that mycoheterotrophy may not be a feature of _Rafflesia_ nor required for its germination/infection.
With this research, we take one step closer to understanding the seed biology of _Rafflesia_ and recalcitrant parasitic plants in general. Developing the toolbox and skillsets necessary for propagating endangered parasitic plants is required to successfully develop conservation and management strategies for these species and their ecosystems. This research provides several tactical guidelines for continuing ongoing efforts to propagate _Rafflesia_. For example, future germination experiments can deprioritize searching for elicitors that are typical of mycoheterotrophy. However, we should prioritize experiments applying laccase and karrikin on conditioned _Rafflesia_ seeds produced from manual pollination. If successful, this approach can also be applied to other difficult to propagate plant species with unique life history traits.
Our research group is dedicated to increase the public's awareness so as to engender engagement in plant and habitat conservation. People are awed every time an _Amorphophallus titanum_ blooms at a public garden because of its remarkable size, appearance, and unpleasant smell. It is a charismatic species that can help people connect with and support the natural world. We believe that _Rafflesia_ has significant untapped engagement potential. We hope this seed transcriptome study brings us closer to a reality in which botanic gardens all over the world grow _Rafflesia_ for conservation, while increasing public awareness and appreciation of an evolutionary marvel—the world's largest flower.
##  AUTHOR CONTRIBUTIONS
Jeanmaire Molina conceived the project, collected samples, and conducted the experiments. Ronniel D. Pedales, Danilo Tandang, and William McLaughlin collected samples with Jeanmaire Molina. Zoé Joly-Lopez performed experiments. Adhityo Wicaksono, Todd P. Michael, Su-Hwan Kwak, Ronniel D. Pedales, Semar Petrus, Allen Mamerto, Brian Tomek, Sumaya Ahmed, Venkatasivasankar Maddu, Kristina Yakubova, and Jeanmaire Molina analyzed the data. Jeanmaire Molina wrote the manuscript with help from Adhityo Wicaksono, Todd P. Michael, Su-Hwan Kwak, Zoé Joly-Lopez, Joseph W. Morin, So-Yon Park, Hyun-Oh Lee, Kyle Wallick, James Adams, Ari Novy, Susan Pell, and Michael D. Purugganan.
##  ACKNOWLEDGMENTS
We are most grateful to our collaborators in the Philippines, to our field guides and assistants, to the Miagao municipality headed by Richard Garin and Macario Napulan, and to their very helpful staff: Jane Branton, Mark Moruga, Isidro Mosura, and so forth. We thank Julie Barcelona, Marites Muyong and family, Auring Nopat and family, Tigmalapad Brgy. Cap. Janette Momay, Kagawad Raymond Nugara, DENR's Nermalie Lita, Josefina de Leon, Anson Tagtag, Giselle Garcia, Nato Andraje, Hector Garrido, Llane Orale, Edgardo Ferrer, Livino Duran, Raul Lorilla, Faith of R6 Penro, Pola Bumanglag, Teresita Paderna, Jojie Gereza, Hiro Masuda, Maricar Escuela, former RD Jim Sampulna, former BMB director Mundita Lim, BPI's Lea Gella Blancaflor, PNM's Ana Labrador, Tito Evangelista, JohnRey Callado, Mike Galindon, Jhaydee Pascual, LIU's Lynn Cabralis, Marni Goldner, Angela Stark, Rick Nader, Randy Burd, Michael Marino, Barbara Neal, Jared Littman, USBG's Stephen Jones, and Mick Erickson, the very helpful USDA staff, and the Bio faculty of University of the Philippines-Diliman with special mention to RDP's mentor, Ian Fontanilla, UP scientists Gisela Concepcion, Arturo Lluisma, and to all who have contributed one way or another. Sofi Mursidawati and Hans Bänziger have been a constant source of inspiration to JM and AW in _Rafflesia_ conservation. We also thank James Westwood and the reviewers for their helpful comments. JM's Indiegogo funders in 2014 have seeded this project (pun intended). Some of this work was also funded by National Science Foundation PBI award #2204938 to JM. This paper is dedicated to the memory of eminent Filipino botanist, Leonard Co.
##  CONFLICT OF INTEREST STATEMENT
The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.
## Open Research
# DATA AVAILABILITY STATEMENT
The datasets generated for this study can be found in NCBI SRA under BioProject PRJNA879908.
##  Supporting Information
Filename | Description  
---|---  
ppp310370-sup-0001-Dataset S1.xlsxExcel 2007 spreadsheet , 3 MB |  **Data S1.** Supporting Information  
Please note: The publisher is not responsible for the content or functionality of any supporting information supplied by the authors. Any queries (other than missing content) should be directed to the corresponding author for the article.
## 
REFERENCES
  * Arekal, G. D., & Shivamurthy, G. R. (1976). Seed germination in _Balanophora abbreviata_. _Phytomorphology_ , 26(2), 135–138. 
Web of Science® Google Scholar
  * Atera, E. A., Itoh, K., & Onyango, J. C. (2011). Evaluation of ecologies and severity of Striga weed on rice in sub-Saharan Africa. _Agriculture and Biology Journal of North America_ , 2(5), 752–760. <https://doi.org/10.5251/abjna.2011.2.5.752.760>
10.5251/abjna.2011.2.5.752.760Google Scholar
  * Bänziger, H. (2004). Studies on hitherto unknown fruits and seeds of some Rafflesiaceae, and a method to manually pollinate their flowers for research and conservation. _Linzer Biologische Beitraege_ , 36, 1175–1198. 
Google Scholar
  * Bao, Y. Z., Yao, Z. Q., Cao, X. L., Peng, J. F., Xu, Y., Chen, M. X., & Zhao, S. F. (2017). Transcriptome analysis of _Phelipanche aegyptiaca_ seed germination mechanisms stimulated by fluridone, TIS108, and GR24. _PLoS ONE_ , 12(11), e0187539. <https://doi.org/10.1371/journal.pone.0187539>
10.1371/journal.pone.0187539PubMed Web of Science® Google Scholar
  * Barcelona, J. F., Pelser, P. B., Balete, D. S., & Co, L. L. (2009). Taxonomy, ecology, and conservation status of Philippine _Rafflesia_ (Rafflesiaceae). _Blumea_ , 54(1–3), 77–93. <https://doi.org/10.3767/000651909x474122>
10.3767/000651909X474122Web of Science® Google Scholar
  * Bascos, E. M. A., Fernando, E. S., Duya, M. V., & Rodriguez, L. J. V. (2021). Beginnings of a plant parasite: Early development of _Rafflesia consueloae_ inside its Tetrastigma host. _Planta_ , 254(3), 61. <https://doi.org/10.1007/s00425-021-03710-4>
10.1007/s00425-021-03710-4CAS PubMed Web of Science® Google Scholar
  * Baskin, J. M., & Baskin, C. C. (2022). Germination and seed/embryo size in holoparasitic flowering plants with “dust seeds” and an undifferentiated embryo. _Botanical Review_ , 88(1), 1–49. <https://doi.org/10.1007/s12229-020-09242-y>
10.1007/s12229-020-09242-yWeb of Science® Google Scholar
  * Bleischwitz, M., Albert, M., Fuchsbauer, H.-L., & Kaldenhoff, R. (2010). Significance of Cuscutain, a cysteine protease from _Cuscuta reflexa_ , in host-parasite interactions. _BMC Plant Biology_ , 10(1), 227. <https://doi.org/10.1186/1471-2229-10-227>
10.1186/1471-2229-10-227PubMed Web of Science® Google Scholar
  * Bouman, F., & Meijer, W. (1994). Comparative structure of ovules and seeds in Rafflesiaceae. _Plant Systematics and Evolution_ , 193(1), 187–212. <https://doi.org/10.1007/BF00983550>
10.1007/BF00983550Web of Science® Google Scholar
  * Bravo, A., Brands, M., Wewer, V., Dörmann, P., & Harrison, M. J. (2017). Arbuscular mycorrhiza-specific enzymes FatM and RAM2 fine-tune lipid biosynthesis to promote development of arbuscular mycorrhiza. _New Phytologist_ , 214(4), 1631–1645. <https://doi.org/10.1111/nph.14533>
10.1111/nph.14533CAS PubMed Web of Science® Google Scholar
  * Cai, L., Arnold, B. J., Xi, Z., Khost, D. E., Patel, N., Hartmann, C. B., Manickam, S., Sasirat, S., Nikolov, L. A., Mathews, S., Sackton, T. B., & Davis, C. C. (2021). Deeply altered genome architecture in the endoparasitic flowering plant _Sapria himalayana_ Griff. (Rafflesiaceae). _Current Biology_ , 31(5), 1002–1011.e9. <https://doi.org/10.1016/j.cub.2020.12.045>
10.1016/j.cub.2020.12.045CAS PubMed Web of Science® Google Scholar
  * Cao, X. L., Li, M. L., Li, J., Song, Y. X., Zhang, X. N., Yang, D. L., Li, M., & Wei, J. (2021). Co-expression of hydrolase genes improves seed germination of _Sinopodophyllum hexandrum_. _Industrial Crops and Products_ , 164, 113414. <https://doi.org/10.1016/j.indcrop.2021.113414>
10.1016/j.indcrop.2021.113414CAS Web of Science® Google Scholar
  * Carrera-Castano, G., Calleja-Cabrera, J., Pernas, M., Gomez, L., & Onate-Sanchez, L. (2020). An updated overview on the regulation of seed germination. _Plants (Basel)_ , 9(6), 703. <https://doi.org/10.3390/plants9060703>
10.3390/plants9060703CAS PubMed Web of Science® Google Scholar
  * Cui, S., Wada, S., Tobimatsu, Y., Takeda, Y., Saucet, S. B., Takano, T., Umezawa, T., Shirasu, K., & Yoshida, S. (2018). Host lignin composition affects haustorium induction in the parasitic plants _Phtheirospermum japonicum_ and _Striga hermonthica_. _New Phytologist_ , 218(2), 710–723. <https://doi.org/10.1111/nph.15033>
10.1111/nph.15033CAS PubMed Web of Science® Google Scholar
  * Davis, C. C., Latvis, M., Nickrent, D. L., Wurdack, K. J., & Baum, D. A. (2007). Floral gigantism in Rafflesiaceae. _Science_ , 315(5820), 1812. <https://doi.org/10.1126/science.1135260>
10.1126/science.1135260CAS PubMed Web of Science® Google Scholar
  * Debeaujon, I., Nesi, N., Perez, P., Devic, M., Grandjean, O., Caboche, M., & Lepiniec, L. (2003). Proanthocyanidin-accumulating cells in Arabidopsis testa: Regulation of differentiation and role in seed development. _Plant Cell_ , 15(11), 2514–2531. <https://doi.org/10.1105/tpc.014043>
10.1105/tpc.014043CAS PubMed Web of Science® Google Scholar
  * Demarsy, E., Buhr, F., Lambert, E., & Lerbs-Mache, S. (2012). Characterization of the plastid-specific germination and seedling establishment transcriptional programme. _Journal of Experimental Botany_ , 63(2), 925–939. <https://doi.org/10.1093/jxb/err322>
10.1093/jxb/err322CAS PubMed Web of Science® Google Scholar
  * Dinelli, G., Bonetti, A., & Tibiletti, E. (1993). Photosynthetic and accessory pigments in _Cuscuta campestris_ Yuncker and some host species. _Weed Research_ , 33(3), 253–260. <https://doi.org/10.1111/j.1365-3180.1993.tb01939.x>
10.1111/j.1365-3180.1993.tb01939.xCAS Web of Science® Google Scholar
  * Dorone, Y., Boeynaems, S., Jin, B., Bossi, F., Flores, E., Lazarus, E., Michiels, E., De Decker, M., Baatsen, P., Holehouse, A. S., & Sukenik, S. (2020). Hydration-dependent phase separation of a prion-like protein regulates seed germination during water stress. SSRN Electronic Journal. <https://doi.org/10.2139/ssrn.3680875>
10.2139/ssrn.3680875Google Scholar
  * Götz, S., Garcia-Gomez, J. M., Terol, J., Williams, T. D., Nagaraj, S. H., Nueda, M. J., Robles, M., Talón, M., Dopazo, J., & Conesa, A. (2008). High-throughput functional annotation and data mining with the Blast2GO suite. _Nucleic Acids Research_ , 36(10), 3420–3435. <https://doi.org/10.1093/nar/gkn176>
10.1093/nar/gkn176CAS PubMed Web of Science® Google Scholar
  * Haughn, G., & Chaudhury, A. (2005). Genetic analysis of seed coat development in _Arabidopsis_. _Trends in Plant Science_ , 10(10), 472–477. <https://doi.org/10.1016/j.tplants.2005.08.005>
10.1016/j.tplants.2005.08.005CAS PubMed Web of Science® Google Scholar
  * Hu, H., Xiong, L., & Yang, Y. (2005). Rice _SERK1_ gene positively regulates somatic embryogenesis of cultured cell and host defense response against fungal infection. _Planta_ , 222(1), 107–117. <https://doi.org/10.1007/s00425-005-1534-4>
10.1007/s00425-005-1534-4CAS PubMed Web of Science® Google Scholar
  * Jia, L., Wu, Q., Ye, N., Liu, R., Shi, L., Xu, W., Zhi, H., Rahman, A. R. B., Xia, Y., & Zhang, J. (2012). Proanthocyanidins inhibit seed germination by maintaining a high level of abscisic acid in _Arabidopsis thaliana_. _Journal of Integrative Plant Biology_ , 54(9), 663–673. <https://doi.org/10.1111/j.1744-7909.2012.01142.x>
10.1111/j.1744-7909.2012.01142.xCAS PubMed Web of Science® Google Scholar
  * Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., & Tanabe, M. (2016). KEGG as a reference resource for gene and protein annotation. _Nucleic Acids Research_ , 44, D457–D462. <https://doi.org/10.1093/nar/gkv1070>
10.1093/nar/gkv1070CAS PubMed Web of Science® Google Scholar
  * Kawasaki, T., Koita, H., Nakatsubo, T., Hasegawa, K., Wakabayashi, K., Takahashi, H., Umemura, K., Umezawa, T., & Shimamoto, K. (2006). Cinnamoyl-CoA reductase, a key enzyme in lignin biosynthesis, is an effector of small GTPase Rac in defense signaling in rice. _Proceedings of the National Academy of Sciences of the United States of America_ , 103(1), 230–235. <https://doi.org/10.1073/pnas.0509875103>
10.1073/pnas.0509875103CAS PubMed Web of Science® Google Scholar
  * Khalaf, E. M., & Raizada, M. N. (2016). Taxonomic and functional diversity of cultured seed associated microbes of the cucurbit family. _BMC Microbiology_ , 16(1), 131. <https://doi.org/10.1186/s12866-016-0743-2>
10.1186/s12866-016-0743-2PubMed Web of Science® Google Scholar
  * Li, S. F., Allen, P. J., Napoli, R. S., Browne, R. G., Pham, H., & Parish, R. W. (2020). MYB-bHLH-TTG1 regulates _Arabidopsis_ seed coat biosynthesis pathways directly and indirectly via multiple tiers of transcription factors. _Plant and Cell Physiology_ , 61(5), 1005–1018. <https://doi.org/10.1093/pcp/pcaa027>
10.1093/pcp/pcaa027CAS PubMed Web of Science® Google Scholar
  * Li, Y. Y., Meng, Z. X., Zhang, Y., Guo, S. X., & Lee, Y. I. (2019). Embryology of _Anoectochilus roxburghii_ : Seed and embryo development. _Botanical Studies_ , 60(1), 6. <https://doi.org/10.1186/s40529-019-0254-1>
10.1186/s40529-019-0254-1PubMed Web of Science® Google Scholar
  * Liu, S. S., Chen, J., Li, S. C., Zeng, X., Meng, Z. X., & Guo, S. X. (2015). Comparative transcriptome analysis of genes involved in GA-GID1-DELLA regulatory module in symbiotic and asymbiotic seed germination of _Anoectochilus roxburghii_ (Wall.) Lindl. (Orchidaceae). _International Journal of Molecular Sciences_ , 16(12), 30190–30203. <https://doi.org/10.3390/ijms161226224>
10.3390/ijms161226224CAS PubMed Web of Science® Google Scholar
  * Miray, R., Kazaz, S., To, A., & Baud, S. (2021). Molecular control of oil metabolism in the endosperm of seeds. _International Journal of Molecular Sciences_ , 22(4), 1621. <https://doi.org/10.3390/ijms22041621>
10.3390/ijms22041621CAS PubMed Web of Science® Google Scholar
  * Molina, J., Hazzouri, K. M., Nickrent, D., Geisler, M., Meyer, R. S., Pentony, M. M., Flowers, J. M., Pelser, P., Barcelona, J., Inovejas, S. A., & Uy, I. (2014). Possible loss of the chloroplast genome in the parasitic flowering plant _Rafflesia lagascae_ (Rafflesiaceae). _Molecular Biology and Evolution_ , 31(4), 793–803. <https://doi.org/10.1093/molbev/msu051>
10.1093/molbev/msu051CAS PubMed Web of Science® Google Scholar
  * Molina, J., McLaughlin, W., Wallick, K., Pedales, R., Marius, V. M., Tandang, D. N., Damatac, A., Stuhr, N., Pell, S. K., Lim, T. M., & Novy, A. (2017). Ex situ propagation of Philippine _Rafflesia_ in the United States: Challenges and prospects. _Sibbaldia_ , 15, 77–96. <https://doi.org/10.24823/Sibbaldia.2017.224>
10.24823/Sibbaldia.2017.224Google Scholar
  * Molina, J., Nikolic, D., Jeevarathanam, J. R., Abzalimov, R., Park, E.-J., Pedales, R., Mojica, E. R. E., Tandang, D., McLaughlin, W., Wallick, K., Adams, J., Novy, A., Pell, S. K., van Breemen, R., & Pezzuto, J. M. (2021). Living with a giant, flowering parasite: Metabolic differences between _Tetrastigma loheri_ Gagnep.(Vitaceae) shoots uninfected and infected with _Rafflesia_ (Rafflesiaceae) and potential applications for propagation. _Planta_ , 255(1), 4–12. <https://doi.org/10.1007/s00425-021-03787-x>
10.1007/s00425-021-03787-xPubMed Web of Science® Google Scholar
  * Mursidawati, S., Ngatari, N., Irawati, I., Cardinal, S., & Kusumawati, R. (2015). Ex situ conservation of _Rafflesia patma_ blume (Rafflesiaceae): An endangered emblematic parasitic species from Indonesia. _Sibbaldia_ , 13, 99–110. <https://doi.org/10.24823/Sibbaldia.2015.77>
10.24823/Sibbaldia.2015.77Google Scholar
  * Nais, J. (2001). Rafflesia of the world. Sabah Parks. 
Google Scholar
  * Nelson, S. K., Ariizumi, T., & Steber, C. M. (2017). Biology in the dry seed: Transcriptome changes associated with dry seed dormancy and dormancy loss in the _Arabidopsis_ GA-insensitive sleepy1-2 mutant. _Frontiers in Plant Science_ , 8, 2158. <https://doi.org/10.3389/fpls.2017.02158>
10.3389/fpls.2017.02158PubMed Web of Science® Google Scholar
  * Ng, S. M., Lee, X. W., Mat-Isa, M. N., Aizat-Juhari, M. A., Adam, J. H., Mohamed, R., Wan, K. L., & Firdaus-Raih, M. (2018). Comparative analysis of nucleus-encoded plastid-targeting proteins in _Rafflesia cantleyi_ against photosynthetic and non-photosynthetic representatives reveals orthologous systems with potentially divergent functions. _Scientific Reports_ , 8(1), 17258. <https://doi.org/10.1038/s41598-018-35173-1>
10.1038/s41598-018-35173-1PubMed Web of Science® Google Scholar
  * Nickrent, D. L. (2020). Parasitic angiosperms: How often and how many? _Taxon_ , 69(1), 5–27. <https://doi.org/10.1002/tax.12195>
10.1002/tax.12195Web of Science® Google Scholar
  * Olszewski, M., Dilliott, M., Garcia-Ruiz, I., Bendarvandi, B., & Costea, M. (2020). _Cuscuta_ seeds: Diversity and evolution, value for systematics/identification and exploration of allometric relationships. _PLoS ONE_ , 15(6), e0234627. <https://doi.org/10.1371/journal.pone.0234627>
10.1371/journal.pone.0234627 updates 
This document has been updated
Click for further information.
CAS PubMed Web of Science® Google Scholar
  * Parthasarathy, A., Savka, M. A., & Hudson, A. O. (2019). The synthesis and role of beta-alanine in plants. _Frontiers in Plant Science_ , 10, 921. <https://doi.org/10.3389/fpls.2019.00921>
10.3389/fpls.2019.00921PubMed Web of Science® Google Scholar
  * Pelser, P. B., Nickrent, D. L., & Barcelona, J. F. (2016). Untangling a vine and its parasite: Host specificity of Philippine _Rafflesia_ (Rafflesiaceae). _Taxon_ , 65(4), 739–758. <https://doi.org/10.12705/654.4>
10.12705/654.4Web of Science® Google Scholar
  * Pelser, P. B., Nickrent, D. L., Callado, J. R. C., & Barcelona, J. F. (2013). Mt. Banahaw reveals: The resurrection and neotypification of the name _Rafflesia lagascae_ (Rafflesiaceae) and clues to the dispersal of _Rafflesia_ seeds. _Phytotaxa_ , 131(1), 35–40. <https://doi.org/10.11646/phytotaxa.131.1.6>
10.11646/phytotaxa.131.1.6Web of Science® Google Scholar
  * Pelser, P. B., Nickrent, D. L., van Ee, B. W., & Barcelona, J. F. (2019). A phylogenetic and biogeographic study of _Rafflesia_ (Rafflesiaceae) in the Philippines: Limited dispersal and high island endemism. _Molecular Phylogenetics and Evolution_ , 139, 106555. <https://doi.org/10.1016/j.ympev.2019.106555>
10.1016/j.ympev.2019.106555PubMed Web of Science® Google Scholar
  * Pennisi, E. (2014). When is a plant no longer a plant? <https://www.science.org/content/article/scienceshot-when-plant-no-longer-plant> [Accessed 11 February 2023]. 
Google Scholar
  * Ranjan, A., Ichihashi, Y., Farhi, M., Zumstein, K., Townsley, B., David-Schwartz, R., & Sinha, N. R. (2014). De novo assembly and characterization of the transcriptome of the parasitic weed dodder identifies genes associated with plant parasitism. _Plant Physiology_ , 166(3), 1186–1199. <https://doi.org/10.1104/pp.113.234864>
10.1104/pp.113.234864PubMed Web of Science® Google Scholar
  * Rehker, J., Lachnit, M., & Kaldenhoff, R. (2012). Molecular convergence of the parasitic plant species _Cuscuta reflexa_ and _Phelipanche aegyptiaca_. _Planta_ , 236(2), 557–566. <https://doi.org/10.1007/s00425-012-1626-x>
10.1007/s00425-012-1626-xCAS PubMed Web of Science® Google Scholar
  * Runo, S., & Kuria, E. K. (2018). Habits of a highly successful cereal killer, Striga. _PLoS Pathogens_ , 14(1), e1006731. <https://doi.org/10.1371/journal.ppat.1006731>
10.1371/journal.ppat.1006731PubMed Web of Science® Google Scholar
  * Santhanam, P., Boshoven, J. C., Salas, O., Bowler, K., Islam, M. T., Saber, M. K., van den Berg, G. C., Bar-Peled, M., & Thomma, B. P. (2017). Rhamnose synthase activity is required for pathogenicity of the vascular wilt fungus _Verticillium dahliae_. _Molecular Plant Pathology_ , 18(3), 347–362. <https://doi.org/10.1111/mpp.12401>
10.1111/mpp.12401CAS PubMed Web of Science® Google Scholar
  * Seed Identification Guide. (2018). www.idseed.org [Accessed June 1, 2022]. 
Google Scholar
  * Song, Y., & Zhu, J. (2019). The roles of metabolic pathways in maintaining primary dormancy of _Pinus koraiensis_ seeds. _BMC Plant Biology_ , 19(1), 550. <https://doi.org/10.1186/s12870-019-2167-2>
10.1186/s12870-019-2167-2CAS PubMed Web of Science® Google Scholar
  * Su, C., Liu, H., Wafula, E. K., Honaas, L., de Pamphilis, C. W., & Timko, M. P. (2020). SHR4z, a novel decoy effector from the haustorium of the parasitic weed _Striga gesnerioides_ , suppresses host plant immunity. _New Phytologist_ , 226(3), 891–908. <https://doi.org/10.1111/nph.16351>
10.1111/nph.16351CAS PubMed Web of Science® Google Scholar
  * Teixeira-Costa, L. (2021). A living bridge between two enemies: Haustorium structure and evolution across parasitic flowering plants. _Brazilian Journal of Botany_ , 44, 165–178. <https://doi.org/10.1007/s40415-021-00704-0>
10.1007/s40415-021-00704-0Web of Science® Google Scholar
  * The Arabidopsis Information Resource (TAIR). (2022). www.arabidopsis.org [Accessed multiple times Jan 2021-Sept 2022]. 
Google Scholar
  * The UniProt Consortium. (2021). UniProt: The universal protein knowledgebase in 2021. _Nucleic Acids Research_ , 49, D480–D489. <https://doi.org/10.1093/nar/gkaa1100>
10.1093/nar/gkaa1100PubMed Web of Science® Google Scholar
  * Thorogood, C., Witono, J. R., Mursidawati, S., & Fleischmann, A. (2021). Parasitic plant cultivation: Examples, lessons learned and future directions. _Sibbaldia_ , 21, 109–136. <https://doi.org/10.24823/Sibbaldia.2022.1892>
10.24823/Sibbaldia.2022.1892Google Scholar
  * Tuquet, C., Farineau, N., & Salé, G. (1990). Biochemical composition and photosynthetic activity of chloroplasts from _Striga hermonthica_ and _Striga aspera_ , root parasites of field-grown cereals. _Physiolia Plantarum_ , 78(4), 574–582. <https://doi.org/10.1111/j.1399-3054.1990.tb05244.x>
10.1111/j.1399-3054.1990.tb05244.xCAS Web of Science® Google Scholar
  * Ueda, M., Fujimoto, M., Arimura, S., Murata, J., Tsutsumi, N., & Kadowaki, K. (2007). Loss of the rpl32 gene from the chloroplast genome and subsequent acquisition of a preexisting transit peptide within the nuclear gene in _Populus_. _Gene_ , 402(1–2), 51–56. <https://doi.org/10.1016/j.gene.2007.07.019>
10.1016/j.gene.2007.07.019CAS PubMed Web of Science® Google Scholar
  * Wakabayashi, T., Joseph, B., Yasumoto, S., Akashi, T., Aoki, T., Harada, K., Muranaka, S., Bamba, T., Fukusaki, E., Takeuchi, Y., Yoneyama, K., Muranaka, T., Sugimoto, Y., & Okazawa, A. (2015). Planteose as a storage carbohydrate required for early stage of germination of _Orobanche minor_ and its metabolism as a possible target for selective control. _Journal of Experimental Botany_ , 66(11), 3085–3097. <https://doi.org/10.1093/jxb/erv116>
10.1093/jxb/erv116CAS PubMed Web of Science® Google Scholar
  * Wang, B., Wang, Y., & Li, J. (2017). Strigolactones. In J. Li, C. Li, & S. M. Smith (Eds.), Hormone metabolism and signaling in plants (pp. 327–359). Elsevier. 
10.1016/B978-0-12-811562-6.00010-4Google Scholar
  * Wang, S., Barron, C., Schiefelbein, J., & Chen, J.-G. (2010). Distinct relationships between GLABRA2 and single-repeat R3 MYB transcription factors in the regulation of trichome and root hair patterning in _Arabidopsis_. _New Phytologist_ , 185, 387–400. <https://doi.org/10.1111/j.1469-8137.2009.03067.x>
10.1111/j.1469-8137.2009.03067.xCAS PubMed Web of Science® Google Scholar
  * Wicaksono, A., Mursidawati, S., & Molina, J. (2021). A plant within a plant: Insights on the development of the _Rafflesia_ endophyte within its host. _Botanical Review_ , 87(2), 233–242. <https://doi.org/10.1007/s12229-020-09236-w>
10.1007/s12229-020-09236-wWeb of Science® Google Scholar
  * Wicaksono, A., Mursidawati, S., Sukamto, L. A., & Teixeira da Silva, J. A. (2016). _Rafflesia_ spp.: Propagation and conservation. _Planta_ , 244(2), 289–296. <https://doi.org/10.1007/s00425-016-2512-8>
10.1007/s00425-016-2512-8CAS PubMed Web of Science® Google Scholar
  * Xi, Z., Bradley, R. K., Wurdack, K. J., Wong, K., Sugumaran, M., Bomblies, K., Rest, J. S., & Davis, C. C. (2012). Horizontal transfer of expressed genes in a parasitic flowering plant. _BMC Genomics_ , 13(1), 227. <https://doi.org/10.1186/1471-2164-13-227>
10.1186/1471-2164-13-227CAS PubMed Web of Science® Google Scholar
  * Xi, Z., Wang, Y., Bradley, R. K., Sugumaran, M., Marx, C. J., Rest, J. S., & Davis, C. C. (2013). Massive mitochondrial gene transfer in a parasitic flowering plant clade. _PLoS Genetics_ , 9(2), e1003265. <https://doi.org/10.1371/journal.pgen.1003265>
10.1371/journal.pgen.1003265CAS PubMed Web of Science® Google Scholar
  * Yang, Z., Zhang, Y., Wafula, E. K., Honaas, L. A., Ralph, P. E., Jones, S., Clarke, C. R., Liu, S., Su, C., Zhang, H., Altman, N. S., Schuster, S. C., Timko, M. P., Yoder, J. I., Westwood, J. H., & dePamphilis, C. W. (2016). Horizontal gene transfer is more frequent with increased heterotrophy and contributes to parasite adaptation. _Proceedings of the National Academy of Sciences of the United States of America_ , 113(45), E7010–E7019. <https://doi.org/10.1073/pnas.1608765113>
10.1073/pnas.1608765113CAS PubMed Web of Science® Google Scholar
  * Yang, Z. Z., Wafula, E. K., Honaas, L. A., Zhang, H. T., das, M., Fernandez-Aparicio, M., Huang, K., Bandaranayake, P. C., Wu, B., der, J. P., Clarke, C. R., Ralph, P. E., Landherr, L., Altman, N. S., Timko, M. P., Yoder, J. I., Westwood, J. H., & dePamphilis, C. W. (2015). Comparative transcriptome analyses reveal core parasitism genes and suggest gene duplication and repurposing as sources of structural novelty. _Molecular Biology and Evolution_ , 32(3), 767–790. <https://doi.org/10.1093/molbev/msu343>
10.1093/molbev/msu343CAS PubMed Web of Science® Google Scholar
  * Yao, Z., Tian, F., Cao, X., Xu, Y., Chen, M., Xiang, B., & Zhao, S. (2016). Global transcriptomic analysis reveals the mechanism of _Phelipanche aegyptiaca_ seed germination. _International Journal of Molecular Sciences_ , 17(7), 1139. <https://doi.org/10.3390/ijms17071139>
10.3390/ijms17071139PubMed Web of Science® Google Scholar
  * Ye, S., Shao, Q., & Zhang, A. (2017). Anoectochilus roxburghii: A review of its phytochemistry, pharmacology, and clinical applications. _Journal of Ethnopharmacology_ , 209, 184–202. <https://doi.org/10.1016/j.jep.2017.07.032>
10.1016/j.jep.2017.07.032CAS PubMed Web of Science® Google Scholar
  * Zhao, L., Wang, S., Fu, Y. B., & Wang, H. (2019). _Arabidopsis_ seed stored mRNAs are degraded constantly over aging time, as revealed by new quantification methods. _Frontiers in Plant Science_ , 10, 1764. <https://doi.org/10.3389/fpls.2019.01764>
10.3389/fpls.2019.01764PubMed Web of Science® Google Scholar


## 
Citing Literature
[](https://nph.onlinelibrary.wiley.com/toc/25722611/2025/7/2 "View Parasitic plants, people and planet")
Volume7, Issue2
Parasitic plants, people and planet
March 2025
Pages 448-466
  * ## Figures
  * ## References
  * ## Related
  * ## Information



Close Figure Viewer
Previous FigureNext Figure
Caption
Download PDF
[](https://nph.onlinelibrary.wiley.com/)
     
© 2025 New Phytologist Foundation 
© 2025 New Phytologist Foundation 
##  Additional links 
### About Wiley Online Library
  * Privacy Policy
  * Terms of Use
  * About Cookies
  * Manage Cookies
  * Accessibility
  * Wiley Research DE&I Statement and Publishing Policies


### Help & Support
  * Contact Us
  * Training and Support
  * DMCA & Reporting Piracy


### Opportunities
  * Subscription Agents
  * Advertisers & Corporate Partners


### Connect with Wiley
  * The Wiley Network
  * Wiley Press Room


Copyright © 1999-2025 John Wiley & Sons, Inc or related companies. All rights reserved, including rights for text and data mining and training of artificial intelligence technologies or similar technologies.
[](https://www.wiley.com "Wiley Home Page")
 

## Log in to Wiley Online Library
Email or Customer ID
Password
 Forgot password? 
 NEW USER >   INSTITUTIONAL LOGIN > [](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)

## Change Password
Old Password
New Password
Too Short Weak Medium Strong Very Strong Too Long
####  Your password must have 10 characters or more: 
  * a lower case character, 
  * an upper case character, 
  * a special character 
  * or a digit

Too Short
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
## Password Changed Successfully
Your password has been changed
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
## Create a new account
Email
 Returning user [](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)

## Forgot your password?
Enter your email address below. 
Email
Please check your email for instructions on resetting your password. If you do not receive an email within 10 minutes, your email address may not be registered, and you may need to create a new Wiley Online Library account.
[](https://nph.onlinelibrary.wiley.com/doi/10.1002/ppp3.10370)
## Request Username
Can't sign in? Forgot your username?
Enter your email address below and we will send you your username
Email
Close
If the address matches an existing account you will receive an email with instructions to retrieve your username