 Skip to main content 

An official website of the United States government 
Here's how you know
Here's how you know

**Official websites use .gov**   
A **.gov** website belongs to an official government organization in the United States. 

**Secure .gov websites use HTTPS**   
A **lock** (  Lock Locked padlock icon  ) or **https://** means you've safely connected to the .gov website. Share sensitive information only on official, secure websites. 
### Service Alert: Planned Maintenance beginning July 25th
Most services will be unavailable for 24+ hours starting 9 PM EDT. Learn more about the maintenance.
 ![
                                  NCBI home page
                               ](https://www.ncbi.nlm.nih.gov/)
Log in
  *  Dashboard 
  *  Publications 
  *  Account settings 
  * Log out 


Search…  Search NCBI 
Primary site navigation 
 Search  
Logged in as: 
  *  Dashboard 
  *  Publications 
  *  Account settings 


Log in
[](https://pmc.ncbi.nlm.nih.gov/ "Home")
Search PMC Full-Text Archive Search in PMC 
  *  Advanced Search 
  *  Journal List 
  *  User Guide 

NewTry this search in PMC Beta Search
  *  View on publisher site
  *  Download PDF
  * Add to Collections
  * Cite
  * Permalink
## PERMALINK
Copy


As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health.  
Learn more: PMC Disclaimer |  PMC Copyright Notice 

Mol Biol Evol
. 2014 Jan 23;31(4):793–803. doi: 10.1093/molbev/msu051
  * Search in PMC
  * Search in PubMed
  * View in NLM Catalog
  * Add to search


# Possible Loss of the Chloroplast Genome in the Parasitic Flowering Plant _Rafflesia lagascae_ (Rafflesiaceae)
Jeanmaire Molina
### Jeanmaire Molina
1Department of Biology, Long Island University, Brooklyn
2Center for Genomics and Systems Biology, New York University
Find articles by Jeanmaire Molina
1,2,*, Khaled M Hazzouri
### Khaled M Hazzouri
3Center for Genomics and Systems Biology, NYU Abu Dhabi, Abu Dhabi, United Arab Emirates
Find articles by Khaled M Hazzouri
3, Daniel Nickrent
### Daniel Nickrent
4Department of Plant Biology, Southern Illinois University, Carbondale
Find articles by Daniel Nickrent
4, Matthew Geisler
### Matthew Geisler
4Department of Plant Biology, Southern Illinois University, Carbondale
Find articles by Matthew Geisler
4, Rachel S Meyer
### Rachel S Meyer
2Center for Genomics and Systems Biology, New York University
Find articles by Rachel S Meyer
2, Melissa M Pentony
### Melissa M Pentony
5Computational Genomics Core, Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 
Find articles by Melissa M Pentony
5, Jonathan M Flowers
### Jonathan M Flowers
2Center for Genomics and Systems Biology, New York University
3Center for Genomics and Systems Biology, NYU Abu Dhabi, Abu Dhabi, United Arab Emirates
Find articles by Jonathan M Flowers
2,3, Pieter Pelser
### Pieter Pelser
6School of Biological Sciences, University of Canterbury, Christchurch, New Zealand
Find articles by Pieter Pelser
6, Julie Barcelona
### Julie Barcelona
6School of Biological Sciences, University of Canterbury, Christchurch, New Zealand
Find articles by Julie Barcelona
6, Samuel Alan Inovejas
### Samuel Alan Inovejas
7Electron Microscope Facility, St. Luke’s Medical Center, Quezon City, Philippines
Find articles by Samuel Alan Inovejas
7, Iris Uy
### Iris Uy
8Philippine Genome Center, University of the Philippines, Diliman, Quezon City, Philippines
Find articles by Iris Uy
8, Wei Yuan
### Wei Yuan
2Center for Genomics and Systems Biology, New York University
Find articles by Wei Yuan
2, Olivia Wilkins
### Olivia Wilkins
2Center for Genomics and Systems Biology, New York University
Find articles by Olivia Wilkins
2, Claire-Iphanise Michel
### Claire-Iphanise Michel
1Department of Biology, Long Island University, Brooklyn
Find articles by Claire-Iphanise Michel
1, Selina LockLear
### Selina LockLear
1Department of Biology, Long Island University, Brooklyn
Find articles by Selina LockLear
1, Gisela P Concepcion
### Gisela P Concepcion
8Philippine Genome Center, University of the Philippines, Diliman, Quezon City, Philippines
Find articles by Gisela P Concepcion
8, Michael D Purugganan
### Michael D Purugganan
2Center for Genomics and Systems Biology, New York University
3Center for Genomics and Systems Biology, NYU Abu Dhabi, Abu Dhabi, United Arab Emirates
Find articles by Michael D Purugganan
2,3,*
  * Author information
  * Article notes
  * Copyright and License information


1Department of Biology, Long Island University, Brooklyn
2Center for Genomics and Systems Biology, New York University
3Center for Genomics and Systems Biology, NYU Abu Dhabi, Abu Dhabi, United Arab Emirates
4Department of Plant Biology, Southern Illinois University, Carbondale
5Computational Genomics Core, Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 
6School of Biological Sciences, University of Canterbury, Christchurch, New Zealand
7Electron Microscope Facility, St. Luke’s Medical Center, Quezon City, Philippines
8Philippine Genome Center, University of the Philippines, Diliman, Quezon City, Philippines
✉
***Corresponding author:** E-mail: jeanmaire.molina@liu.edu; mp132@nyu.edu.
**Associate editor:** Brandon Gaut
Issue date 2014 Apr.
© The Author 2014. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.
This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<http://creativecommons.org/licenses/by-nc/3.0/>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com
PMC Copyright notice
PMCID: PMC3969568 PMID: 24458431
##  Abstract
_Rafflesia_ is a genus of holoparasitic plants endemic to Southeast Asia that has lost the ability to undertake photosynthesis. With short-read sequencing technology, we assembled a draft sequence of the mitochondrial genome of _Rafflesia lagascae_ Blanco, a species endemic to the Philippine island of Luzon, with ∼350× sequencing depth coverage. Using multiple approaches, however, we were only able to identify small fragments of plastid sequences at low coverage depth (<2×) and could not recover any substantial portion of a chloroplast genome. The gene fragments we identified included photosynthesis and energy production genes (_atp_ , _ndh_ , _pet_ , _psa_ , _psb_ , _rbcL_), ribosomal RNA genes (_rrn16_ , _rrn23_), ribosomal protein genes (_rps7_ , _rps11_ , _rps16_), transfer RNA genes, as well as _matK_ , _accD_ , _ycf2_ , and multiple nongenic regions from the inverted repeats. None of the identified plastid gene sequences had intact reading frames. Phylogenetic analysis suggests that ∼33% of these remnant plastid genes may have been horizontally transferred from the host plant genus _Tetrastigma_ with the rest having ambiguous phylogenetic positions (<50% bootstrap support), except for _psaB_ that was strongly allied with the plastid homolog in _Nicotiana_. Our inability to identify substantial plastid genome sequences from _R. lagascae_ using multiple approaches—despite success in identifying and developing a draft assembly of the much larger mitochondrial genome—suggests that the parasitic plant genus _Rafflesia_ may be the first plant group for which there is no recognizable plastid genome, or if present is found in cryptic form at very low levels.
**Keywords:** holoparasite, _Tetrastigma_ , plastid, gene loss, horizontal gene transfer, NUPTs
##  Introduction
The ability to conduct photosynthesis is one of the defining features of plants. They owe this capacity to endosymbiotic chloroplasts, once free-living cyanobacteria that were assimilated by an ancestral protist ∼1.5 billion years ago (Gould et al. 2008). Chloroplasts, one of several plastids that develop from meristematic proplastids, are the site of production and storage of key plant metabolites (Wise 2006). Plastid organelles—which include chromoplasts and amyloplasts—are also involved in fatty acid synthesis, production of tetrapyrroles and aromatic substances, and pigment and starch storage (Neuhaus and Emes 2000).
Chloroplasts possess circular DNA genomes, which range size from ∼107 to 217 kb (mean of ∼152 kb) in the 220 photosynthetic angiosperms that have been examined to date (<http://www.ncbi.nlm.nih.gov/genome>, last accessed February 8, 2014). Chloroplast genomes (or plastomes) typically encode ∼85 proteins and ∼45 transfer RNA (tRNA) and ribosomal RNA (rRNA). The genome is a relic of the endosymbiotic origin of this organelle and is reduced in size from its bacterial ancestor, having lost some genes as well as transferring others to the nucleus (Martin 2003). As a consequence of this evolutionary relocation of genes, most proteins required for chloroplast function (∼2,500–3,500) are encoded by nuclear loci (Blanchard and Schmidt 1995).
Chloroplast genome structure is highly conserved across flowering plants, with the interesting exception of parasitic plants. Plant parasitism is an interesting evolutionary adaptation, arising independently at least 12–13 times in flowering plants, with about 1% of all known angiosperm species being parasitic plants (Barkman et al. 2007; Westwood et al. 2010). Hemiparasites, which depend on their hosts only for water and inorganic nutrients, still retain much of their chloroplast genomes (Braukmann and Stefanovic 2012). Achlorophyllous holoparasites, however, have undergone evolutionary reduction in genome size associated with gene loss. _Epifagus virginiana_ , for example, has a much smaller plastid genome (∼70 kb) and is about half of the expected plastome size in autotrophic land plants (Wolfe et al. 1992). This reduced chloroplast (or plastid) genome contains only 42 genes and has lost the loci for photosynthesis and chlororespiration (Wolfe et al. 1992). Despite these reductions in genome size, all plants examined to date, including the nonphotosynthetic parasitic plants, as well as apicomplexan parasites such as _Plasmodium_ , continue to retain even a vestige of a plastid genome (McFadden et al. 1996; Maréchal and Cesbron-Delauw 2001; Krause 2008; Li et al. 2013).
The genus _Rafflesia_ , which belongs to the family Rafflesiaceae (order Malpighiales), is one of the eight known genera of plant holoparasites. _Rafflesia_ is unique to the tropics of Southeast Asia, with some species in the genus producing the largest single flowers in the world, growing up to a meter in diameter. It has no stems, roots, or leaves, with only its massive flower protruding from the roots or stems of its sole host plant, the tropical vine _Tetrastigma_ (Vitaceae) (Nais 2001) (see fig. 1). Nearly one-third of the 30 known _Rafflesia_ species are endemic to the Philippines (Nickrent et al. 1997; Barcelona et al. 2009). Other members of the Rafflesiaceae family include _Sapria_ and _Rhizanthes_ , which are also holoparasites of _Tetrastigma_ (Nais 2001). 
### Fig. 1.
[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3969568_msu051f1p.jpg)
Open in a new tab
Open flower of _R. lagascae_ Blanco. The flower is 15–20 cm in diameter.
Attempts to isolate highly conserved plastid genes from members of the holoparasite genus _Rafflesia_ (Rafflesiaceae) (Nickrent et al. 1997; Davis et al. 2007) have failed, and another study has indicated the possibility of plastid genome loss in _Rafflesia leonardi_ (Nickrent DL, Molina J, Geisler M, Bamber AR, Pelser PB, Barcelona JF, Inovejas SAB, Uy I, Purugganan MD, unpublished data). Here we provide a strong evidence that suggests that the chloroplast/plastid genome is entirely absent in _R_ _. lagascae_ Blanco (see fig. 1), a species found only on the Philippine island of Luzon but nevertheless the most widespread of all the Philippine _Rafflesia_ species (Pelser et al. 2013). With data from Illumina next-generation sequencing, we employ multiple separate techniques for organellar genome assembly. We are able to assemble a draft of the _R. lagascae_ mitochondrial genome at high coverage. We cannot, however, identify an intact plastid genome in _R. lagascae_ , which indicates that members of the parasitic plant genus _Rafflesia_ may be the first plant group shown to have lost its plastid genome.
##  Results
### Draft Sequence Assembly of a Mitochondrial Genome in _Rafflesia_
A floral bud of _R. lagascae_ was collected from Cagayan province in the Philippines. Attached only at its base to its _Tetrastigma_ host, _Rafflesia_ tissue was carefully dissected from the host plant, and genomic DNA was extracted from the disk distant from host tissue and enclosed in layers of bracts. Both 100-bp and 3-kb insert libraries were made from genomic DNA and sequenced using Illumina next-generation technology.
Of the approximately 440 million Illumina paired-end (PE) sequencing reads from _R. lagascae_ from both insert libraries, we used two distinct methods to assemble a draft sequence of the mitochondrial genome. First, we used a bait mapping approach by employing previously published data for _R. cantleyi_ (Xi et al. 2013) to assemble the mitochondrion from this species using SOAPdenovo (Luo et al. 2012), and used this assembled _R. cantleyi_ sequence as bait to identify mitochondrial genome sequences from the _R. lagascae_ Illumina reads. These identified reads were then assembled by SOAPdenovo to provide a draft genome assembly of the _R. lagascae_ mitochondrial genome. We were able to identify and assemble ∼320.3 kb of the _R. lagascae_ mitochondrial genome (N50 = 4.45 kb); this constitutes a draft assembly with 213 gaps (see fig. 2). The mean sequencing depth coverage across the reference is 349.7×, with a standard deviation of 173.02× (see supplementary fig. S1, Supplementary Material online, for sequence depth coverage across the draft-assembled genome). 
#### Fig. 2.
[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3969568_msu051f2p.jpg)
Open in a new tab
Draft structure of _R. lagascae_ mitochondrial genome. The gene positions indicated are based on the assumption of synteny with _Ricinus_ (GenBank accession number HQ874649; Rivarola et al. 2011)_._ The coordinates and encoded products of the specific genes are shown in supplementary table S1, Supplementary Material online. Although depicted as a complete circular genome, it should be stressed that this is a draft assembly with 213 gaps and that portions of the mitochondrial sequence remain unassembled.
Additionally, we used a de novo assembly approach (without any reference genome) to obtain ∼1,447,235 sequence contigs of the _R. lagascae_ sequence data using CLC Genomics Workbench (CLC Bio, Aarhus, Denmark), with an N50 = 182 bp. The low N50 for the entire data set is due to either the large size of the _R. lagascae_ nuclear genome or its high repeat content. Despite this, we were able to readily identify ∼4,000 sequence contigs containing mitochondrial genome sequence through Blast analysis of assembled sequence contigs against 15 plant mitochondrial genomes. The largest 49 sequence contigs were shown to have about 210× sequence coverage. These contigs ranged from 1.0 to 17.3 kb, with an average size of 6.3 ± 5.0 kb; this represents a total sequence length of 382 kb.
We identified 42 known plant protein-coding mitochondrial genes, as well as 60 unique open reading frames (ORFs) (see fig. 2 and supplementary table S1, Supplementary Material online). Phylogenetic analyses of the mitochondrial protein-coding genes show that 7 of 24 genes with clear phylogenetic placement (29%) have these _Rafflesia_ loci allied to _Vitis vinifera_ (Vitaceae) or other plant groups, instead of the more closely related _Ricinus communis_ (Euphorbiaceae, Malpighiales) (results not shown)_._ The other identified mitochondrial genes show equivocal placement in the phylogenies. These confirm previous findings of rampant horizontal gene transfer (HGT) in this genus (Xi et al. 2012, 2013; Nickrent DL, Molina J, Geisler M, Bamber AR, Pelser PB, Barcelona JF, Inovejas SAB, Uy I, Purugganan MD, unpublished data).
### Searching for a Plastid Genome
We attempted to recover plastid sequences from _R. lagascae_ using several approaches. First, we mapped the sequence contigs obtained from the CLC Genomics Workbench assembly method onto the chloroplast genomes of _Ricinus_ and _Vitis_ , using the program Geneious R6 (Biomatters, Auckland, New Zealand). This yielded 17 sequence contigs (G1–G17; table 1) that mapped to the plastid genomes of these two species, with a total length of ∼3.9 kb. We also conducted a BlastN search of all conserved plastid genes available from GenBank against the assembled CLC sequence contigs, which identified additional 26 contigs with significant (e-value < 1e−10) hits (table 1). In addition, we generated profile hidden Markov models (HMMs; Henderson et al. 1997) from alignments of conserved plastid genes. This identified three more contigs with significant hits (e-value < 1e−10) (table 1). Together, these approaches identified a total of 46 putative plastid sequence contigs with an average length of 242 bp and a total length of 11.5 kb. 
#### Table 1.
Identified Plastid Sequences in _Rafflesia lagascae_ Including Noncoding Sequences in Inverted Repeat (IR) Regions.
Gene Name | Size (bp) | Method of Recovery | Contig Numbera | Phylogenetic Allianceb  
---|---|---|---|---  
_accD_ | 136 | BlastN | 763,474 |  _Vitis_ mt, cp, nucc  
_atpA_ | 158 | BlastN | 1,105,137 | —  
_atpB_ | 127 | BlastN | 568,017 | —  
IR | 138 | BlastN | 690,602 |  _Hevea_ cp, _Vitis_ cp  
IR | 160 | BlastN | 552,527 | —  
IR | 163 | Geneious | G14 | —  
IR | 190 | Geneious | G11 | —  
IR | 199 | Geneious | G3 |  _Vitis_ cp, nucc  
IR | 207 | BlastN | 435,549 |  _Vitis_ nucc  
IR | 219 | BlastN | 1,355,988 | —  
IR | 227 | Geneious | G15 |  _Vitis_ nucc  
IR | 274 | BlastN | 426,447 |  _Vitis_ cp, nucc  
IR | 299 | Geneious | G6 | —  
IR, _trnA-UGC_ | 104 | Geneious | G8 |  _Vitis_ nuc  
113 | Geneious | G7 |  _Vitis_ nuc  
168 | Geneious | G16 |  _Vitis_ nuc  
330 | BlastN | 859,744 |  _Vitis_ nuc  
484 | Geneious | G10 |  _Vitis_ nucc  
IR, _trnI-GAU_ | 496 | BlastN | 12,131 | —  
IR, _trnV-GAC_ | 145 | Geneious | G13 |  _Vitis_ nuc  
178 | Geneious | G5 |  _Vitis_ nuc  
178 | BlastN | 1,147,528 |  _Vitis_ nucc  
_matK_ | 227 | BlastN | 710,409 |  _Vitis_ cp, mt, nucc  
_ndhB_ | 118 | Geneious | G4 | —  
_ndhJ_ | 247 | BlastN | 1,167,865 |  _Vitis_ cp, nucc  
247 | Geneious | G1 |  _Vitis_ cp, nuc  
_petB_ | 140 | Geneious | G2 | —  
_petG_ | 116 | BlastN | 823,312 | —  
_psaB_ | 211 | BlastN | 76,772 |  _Nicotiana_ cp  
315 | BlastN | 662,630 |  _Nicotiana_ cp  
324 | BlastN | 972,383 |  _Nicotiana_ cpc  
_psbA_ | 161 | BlastN | 1,131,304 |  _Vitis mt_ c  
_psbD_ | 280 | BlastN | 39,796 | —  
_psbZ_ | 137 | BlastN | 1,038,399 | —  
_rbcL1_ | 395 | BlastN | 127,053 | —  
_rbcL2_ | 647 | BlastN | 170,975 |  _Vitis_ cp, mtc  
_rps11_ | 113 | HMM | 322,006 |  _Vitis_ nucc  
_rps16_ | 148 | BlastN | 641,235 | —  
_rps7_ | 119 | HMM | 864,624 | —  
_rrn16 (IR)_ | 191 | Geneious | G12 | —  
203 | BlastN | 1,114,615 | —  
317 | BlastN | 4,358 | —  
_rrn23 (IR)_ | 139 | Geneious | G17 |  _Vitis_ cp  
766 | Geneious | G9 |  _Vitis_ cp  
1026 | BlastN | 19,544 |  _Vitis_ cpc  
_ycf2 (IR)_ | 203 | HMM | 113,164 | —  
Open in a new tab
Note.—Certain IR sequences may appear multiple times because they belong to the inverted repeats region of the plastid genome, whereas genes like _ndhJ_ were identified by different methods.
aContig number is the specific contig out of the approximately 1.4 million contigs from CLC, except for those prefixed with “G,” which were derived from Geneious.
bTaxa and genome/s (mt, mitochondria; cp, chloroplast; nuc, nuclear) to which _Rafflesia_ was shown strongly associated with (>50% BS, compare with fig. 4) in phylogenetic analyses; sequences marked with “—” had ambiguous phylogenetic positions. Multiple taxon/genome associations represent polytomous nodes >50% BS in which _Rafflesia_ is embedded in.
cCorresponding phylogenies for these sequence contigs that show _Rafflesia_ in unequivocal positions are provided as Supplementary Material (taxa in the _rps11_ phylogeny represent the only significant hits recovered).
To ensure that our failure to identify a plastid genome was not due to a problem of the genome assembly method, we identified 925 Illumina 100-bp PE reads (out of ∼214 million reads) that directly mapped to plastid sequences found in GenBank. These sequences were then de novo assembled using SOAPdenovo, another sequence assembly program for short-read sequencing data (Luo et al. 2012). Aside from those sequences already identified by the previous methods, we found additional five sequence fragments, but they appear embedded in the assembled mitochondrial genome of _R. lagascae_.
Unlike the high sequence coverage for the draft assembly of the mitochondrial genome (∼350×), the mean sequence coverage for the plastid sequence fragments we identified was substantially lower at 1.48× ± 1.26× reads. The anomalously low sequencing read depth coverage contrasts with the several hundred copies of plastid genomes that should exist within plant cells (Bock 2007). As a comparison with the normal expectation in different species, we used Illumina whole genome re-sequencing data to demonstrate that chloroplast/plastid genome sequencing depth coverage was nearly equal to or exceeded that of mitochondrial genome in leaf tissue in _Oryza_ , _Phoenix_ _,_ and _Arabidopsis_ , and light-grown single-cell culture in the algae _Chlamydomonas_ (see fig. 3). The sequencing reads in these species covered >98% of the bases in these organellar genome sequences. 
#### Fig. 3.
[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3969568_msu051f3p.jpg)
Open in a new tab
Whole-genome sequencing coverage for mitochondrial and chloroplast genomes in different species. Blue bars are for chloroplast genomes, whereas red bars are for mitochondrial genomes. The tissue source for the genomic DNA is indicated; for _C. reinhardtii_ , it is single-cell culture under constant light.
Because the plastid copy number is reduced in nonphotosynthetic tissues like roots (Isono et al. 1997), it is possible that lower levels of plastid DNA in the nonphotosynthetic _Rafflesia_ may be the source of our inability to identify a plastid genome in this species. As a further positive control, we therefore obtained ∼62 and ∼104 million 100-bp PE Illumina sequencing reads from root DNA in _Oryza glaberrima_ and _Arabidopsis thaliana_ , respectively, and examined relative levels of plastid and mitochondrial genome copies in these nonphotosynthetic tissues. Based on sequencing depth coverage from root genomic DNA (see fig. 3), the plastid genome in _O. glaberrima_ is ∼66% that of mitochondrial genome levels, whereas in _A. thaliana_ the plastid genome is found at ∼3.5-fold greater levels than mitochondrial genome (down from an ∼17-fold greater level in leaves). Our data demonstrate a reduction in plastid genome levels in the nonphotosynthetic root compared with leaf tissues but nevertheless show that plastid genome levels can be substantial in nonphotosynthetic tissues.
### Fragments of the Plastid Genome
Although we could not identify an intact plastid genome, we did find small fragments of plastid genes that ranged in size from 104 to 1,026 bp. We recovered short segments of 17 protein-coding genes (including ribosomal proteins), two rRNA and three tRNA genes, as well as ten intergenic sequences that are found in inverted repeat regions of the chloroplast genome in other species (see table 1). No full gene sequences were identified. None of the plastid sequence contigs had intact reading frames when aligned with coding sequences of plastid genes from photosynthetic taxa (results available upon request).
To determine whether any of the recovered plastid sequences were expressed, we also mapped the sequencing reads from the _R. cantleyi_ transcriptome library (accession SRA052224) to these _R. lagascae_ sequence fragments (Xi et al. 2012). The greatest number of reads mapped were 23 singleton reads from the partial ribosomal rrn16 (∼150 bp) fragment. These results indicate that there is no significant expression for any of these putative plastid-derived sequences, at least in the actively developing floral bud tissue of this parasitic plant.
### Phylogenetic Analysis of Remnant _Rafflesia_ Plastid Gene Sequences
Phylogenetic analysis of the 46 plastid sequence fragments that we identified show that 15 of these _Rafflesia_ sequence fragments are allied with _Vitis_ with >50% BS (table 1, fig. 4 and supplementary data S2, Supplementary Material online), The rest of the plastid sequences show _Rafflesia_ in equivocal positions (BS <50%). 
#### Fig. 4.
[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3969568_msu051f4p.jpg)
Open in a new tab
Phylogenies of recovered plastid sequences from _R. lagascae_. _Rafflesia_ sequences group with (_A_) both nuclear (nuc) and plastid (cp) sequences in _Vitis_ ; (_B_) only _Vitis_ nuclear sequence; (_C_) only _Vitis_ plastid sequences; (_D_) _Nicotiana_ plastid sequence. Only bootstrap support >50% is indicated.
Four sequence fragments, including _ndhJ_ (fig. 4 _A_), depict _Rafflesia_ plastid sequences associated with both _Vitis_ ’ nuclear and plastid sequences. Five other sequences such as a noncoding sequence from one of the inverted repeats were more similar to a _Vitis_ ’ nuclear sequence than they are to plastid sequences (fig. 4 _B_ ; supplementary data S2, Supplementary Material online; table 1). Only _Rafflesia_ ’s _rrn23_ was solely with a _Vitis_ chloroplast sequence (fig. 4 _C_). A 136-bp fragment from the _accD_ gene had _Rafflesia_ grouping with nuclear, plastid and mitochondrial copies (76% BS; supplementary data S2, Supplementary Material online, and table 1).
No plastid sequence was found to be phylogenetically associated with _Ricinus_ or _Hevea_ , the closest relatives to _Rafflesia_ with available organellar genome sequences. However, the _Rafflesia psaB_ gene fragment grouped with the _Nicotiana_ homolog with 80% BS (fig. 4 _D_).
The anomalous phylogenetic placement of these remnant _Rafflesia_ plastid genes may arise from contamination from the DNA of the host plant _Tetrastigma_. To test for contamination of _Tetrastigma_ in the _R. lagascae_ DNA extract, we used barcoding primers to amplify the _rbcL_ gene (Kress et al. 2009) in these two species. We were unable to polymerase chain reaction (PCR)-amplify _rbcL_ from _R. lagascae_ , although this gene was easily amplified from the host _Tetrastigma_ and from other evolutionary divergent photosynthetic taxa from the asterid and rosid families. Interestingly, we were able to recover two nonoverlapping segments of _rbcL_ sequence (_rbcL1_ , _rbcL2_) from the Illumina sequence contigs (∼1 kb in size) from _R. lagascae_ (table 1), but these sequences are diverged in the barcoding primer sequence regions (Kress et al. 2009) that are normally conserved across multiple divergent autotrophic angiosperm taxa. Like the other recovered plastid sequences, _Rafflesia_ ’s _rbcL_ contains premature stop codons.
##  Discussion
### Possible Loss of the Plastid Genome in a Parasitic Plant
The parasitic plant lifestyle affords an intimate connection between a parasite and its host plant. As in many parasites, this can lead to subsequent relaxation of selection pressure to maintain key genes in the parasite as they become dependent on host plants for crucial functions (Bromham et al. 2013; Wicke et al. 2013). Moreover, the close connection to the host can also lead to genetic transfer of information to the parasite (Davis and Wurdack 2004; Xi et al. 2012, 2013).
The evolution of the chloroplast genome in parasitic plants, particularly nonphotosynthetic holoparasites, can lead to significantly reconfigured plastomes (Wicke et al. 2013). In these plants many photosystem and energy production genes are lost from the plastome (Krause 2008; Li et al. 2013; Wicke et al. 2013). The 45.6-kb plastome of _Conopholis americana_ (Orobanchaceae) is the smallest published plastid genome to date (Colwell 1994; Wicke et al. 2013). Though devoid of genes expected to be present in autotrophic plants, it still maintains genes that are conserved in all previously sequenced plastomes, like genes for rRNA, some genes for ribosomal proteins, tRNA (e.g., _trnE_ and _trnfM_) and the essential genes _clpP_ and _ycf2_ (Wicke et al. 2013). Other parasitic plants and even the protist _Plasmodium_ , descendant of the same photosynthetic protist as plants, still show remnant plastid genomes (McFadden et al. 1996; Maréchal and Cesbron-Delauw 2001; Krause 2008; Li et al. 2013).
It is clearly challenging to prove the complete absence of a plastid genome (Keeling 2010). Nevertheless, our inability to identify substantial plastid genome sequences from _R. lagascae_ using multiple approaches, despite success in identifying and developing a draft assembly of the much larger mitochondrial genome, strongly suggests that there may be no recognizable plastid genome in the parasitic plant _R. lagascae_ , or if present in cryptic form, is at very low levels. Moreover, a similar result has also been observed for _R. leonardi_ (Nickrent DL, Molina J, Geisler M, Bamber AR, Pelser PB, Barcelona JF, Inovejas SAB, Uy I, Purugganan MD, unpublished data), which our study now reinforces with multiple lines of evidence. These results together suggest that plastid genome loss may be shared across multiple _Rafflesia_ species, and this putative loss is likely an evolutionary consequence of the very ancient onset of parasitism in the family (Xi et al. 2013), perhaps dating back to mid-Cretaceous (Molina J, unpublished data).
It is possible for organelles to lose their genomes. This has already been documented in some anaerobic ciliates, trichomonads, and fungi, which possess hydrogenosomes, genomeless organelles that produce molecular hydrogen and are derived from mitochondria (Van der Giezen et al. 2005). However, the loss of the plastome has not yet been demonstrated in plants but deemed possible (Palmer 1997). Plastids have been secondarily lost outside the flowering plant lineage, including the protozoan trypanosomes (Martin and Borst 2003). However, in a recent study by Braukmann et al. (2013) on some species of the parasitic flowering plant _Cuscuta_ (Convolvulaceae), difficulty has been reported in detecting plastid rRNA (_rrn_) genes that are highly conserved elements in plant plastomes even in heterotrophic species (Krause 2011; Wicke et al. 2011; Li et al. 2013). Accordingly, Braukmann et al’s (2013, p. 9) observations led them to conclude that some holoparasitic _Cuscuta_ may have “reached the same or similar evolutionary endpoint, where the very presence of a plastid genome is questionable.”
Despite the possible loss of the chloroplast and its genome, there still remain plastid-like structures in _Rafflesia_. Images from ultrastructural analysis from the congeneric _R. philippensis_ using transmission electron microscopy (TEM) demonstrate that _Rafflesia_ (Nickrent DL, Molina J, Geisler M, Bamber AR, Pelser PB, Barcelona JF, Inovejas SAB, Uy I, Purugganan MD, unpublished data; for additional images, see fig. 5) does contain plastid-like compartments with homogeneous stroma (Renzaglia K, personal communication). None of these structures have the distinctive lamellar/endomembrane system found in all types of plastids. These suggest that _Rafflesia_ retains plastid compartments for certain metabolic functions, even in the apparent absence of a plastid genome. 
#### Fig. 5.
[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3969568_msu051f5p.jpg)
Open in a new tab
Electron micrographs of cells from _R. philippensis_ ramenta indicating plastid-like (P) structures (N, nucleus; m, mitochondria; V, vacuole). Scale bars = 1 µm.
There are two possible scenarios for the evolution of these genome-less plastids. One is that any relevant genes still necessary for metabolic function have relocated to the nucleus and/or mitochondria. Another possibility is that these _Rafflesia_ plastids were originally obtained from the host, with subsequent translocation of host-encoded plastid genes to the nucleus and eventually degenerating as pseudogenes. The latter possibility may explain the large proportion of remnant nonfunctional plastid gene sequences in _R. lagascae_ that are phylogenetically allied with _Vitis_ nuclear sequences (table 1; supplementary data S2, Supplementary Material online). Such host-derived plastids have been observed in certain parasitic red algae (Goff and Coleman 1995) as well as in natural grafts of sexually incompatible species of _Nicotiana_ (Stegemann et al. 2012). These two possibilities are not mutually exclusive, and the precise genetic basis for the maintenance of these _Rafflesia_ plastid-like structures must await more detailed analysis of fully assembled nuclear genomes.
Interestingly, osmiophilic plastoglobules or carotenoid bodies, which are typical inclusions of chromoplasts in colored flowers (Camara et al. 1995), were also not seen in the _Raffesia_ plastid-like structures, and thus, may not be the primary source of the bright red-orange coloration characteristic of _Rafflesia_ species. Instead, there were very large vacuoles observable in _Rafflesia_ ramenta filled with osmiophilic material, which may be phenolic compounds or terpenoids that also appear as electron-dense material in other TEM studies of trichomes (Sacchetti et al. 1999; Wen-Zhe et al. 2002). These plant terpenoids can serve as precursors for a diversity of plant metabolites such as carotenoid and anthocyanin pigments and volatiles responsible for flower color and odor (Tanaka et al. 2008).
Several hypotheses have been proposed with respect to what biochemical constraints on plastid genome size or its complete loss might exist owing to the need to retain particular genes required for metabolism (Bungard 2004; Barbrook et al. 2006). The essential tRNA hypothesis states that plastid-encoded _trnE_ is considered essential for heme biosynthesis (a component of the mitochondrial P450 cytochromes) and could not be easily replaced by a cytosolic tRNA. It may be that _Rafflesia_ continues to retain plastids for various metabolic functions, but the genes that encode for these are found in the nucleus or in the mitochondria as in the case of a _trnE_ sequence recovered in _R. leonardi_ (Nickrent DL, Molina J, Geisler M, Bamber AR, Pelser PB, Barcelona JF, Inovejas SAB, Uy I, Purugganan MD, unpublished data). The fates of other essential plastid genes, such as the _clpP_ and _ycf2_ loci (Wicke et al. 2013), are unknown, and must await detailed genomic and biochemical studies on these parasitic plant species.
### Vestiges of the Missing Plastid Genome
The small regions we could successfully identify in _R. lagascae_ as putative plastid sequences represent less than 10% of the chloroplast genome of photosynthetic plants, and do not have intact reading frames and so are likely nonfunctional. These results would make _Rafflesia_ arguably the plant with the smallest amount of plastid “genome” sequence that has been observed thus far (Krause 2008; Li et al. 2013). Given the low sequencing read coverage for these gene fragments (<2× coverage), it is likely that these remnant plastid sequences are located in the nuclear genome, similar to what has been observed for nuclear-integrated chloroplast genes and/or nuclear plastid DNAs (NUPTs) observed in other plant species (Blanchard and Schmidt 1995; Kleine et al. 2009).
Phylogenetic analyses of the remnant plastid gene fragments in _Rafflesia_ show that many of these genes have anomalous but strongly supported (>50% BS support) phylogenetic placements that suggest they are the products of HGT. Most of these genes grouped phylogenetically with homologs in _Vitis_ (which belongs to the same family as the host species _Tetrastigma_), rather than either _Ricinus_ or _Hevea_ , which together are more closely related to _Rafflesia_. There is a possibility that these may be due to contamination of the _Rafflesia_ DNA, but control experiments using universal _rbcL_ barcoding primers suggest that this is unlikely. The anomalous phylogenetic placement of remnant _Rafflesia_ plastid sequences may also arise from a complex history of gene duplication and extinction, although we think that HGT is a more parsimonious explanation.
Although these other possibilities cannot be completely ruled out, we feel that our results suggest that ∼33% of the plastid loci we have identified in _R. lagascae_ may have been horizontally acquired, most likely from its host as a result of parasitism. It has already been demonstrated in other parasitic plants that the plasmodesmatal continuity between host and parasite allows for molecular movement, including that of genetic material (Birschwilks et al. 2006; Roney et al. 2007; Talianova and Janousek 2011). Rampant HGT between _Rafflesia_ and _Tetrastigma_ involving several nuclear and mitochondrial genes has also been repeatedly shown (Xi et al. 2012, 2013). Individual phylogenies of chloroplast sequences recovered from the confamilial _Sapria_ have also exhibited a closer relationship to _Vitis_ than to _Ricinus_ (Xi et al. 2013). A complete genome sequence of the _Rafflesia_ nuclear genome will be necessary in order to conclusively determine the fate of the genes that would normally reside in the plastid as well as examine the other molecular evolutionary consequences of the obligate parasitic lifestyle of this enigmatic yet fascinating plant genus.
##  Materials and Methods
### Illumina Whole _-_ Genome Sequencing
A flower bud of _R. lagascae_ was collected from Sitio Kanapawan, Barangay Bolos Point, Gattaran Municipality, Cagayan Province, Philippines in 2010 (collection number Nickrent 5791). All necessary collecting (Philippine Department of Environment and Natural Resources Gratuitous Permit 193, and subsequent renewals GP 202 and 217), transport, and export permits were obtained.
A sizeable _Rafflesia_ bud, attached only at its base to its host, _Tetrastigma_ , was carefully dissected from the host plant. Genomic DNA was extracted from a portion of the disk, which is sufficiently distant from host tissue and enclosed in layers of bracts. DNA extraction was performed following Nickrent et al. (2004). Contamination of _R. lagascae_ DNA by _Tetrastigma_ was tested using standard PCR amplification with degenerate _rbcL_ barcoding primers (Kress et al. 2009). PCR amplification was positive in _Tetrastigma_ but negative in _R_. _lagascae_ (as well as in _R. leonardi_). The extracted _R. lagascae_ DNA was submitted to Ambry Genetics (Aliso Viejo, CA) for Illumina next-generation sequencing, using both a 100-bp and a 3-kb insert size library on Illumina HiSeq 2000. Sequencing reads are deposited with the NCBI Sequence Read Archive (SRX434531).
### Mitochondrial Genome Assembly and Analysis
Illumina sequencing reads from _R. lagascae_ were assembled using two approaches. In one approach, we used a combination of SOAPdenovo (Luo et al. 2012) and reference-assisted mapping to assemble _R. lagascae_ ’s organellar genomes. We first assembled the mitochondrial gene sequence of the closely related species, _R. cantleyi_ (Xi et al. 2013), whose Illumina reads were previously published and available in NCBI (accession SRR629613) and used this assembly of its mitochondrial genome as reference bait sequence. We then collected all the _R. lagascae_ 100-bp PE reads that mapped to the _R. cantleyi_ mitochondrial genome and used these to de novo assemble _R. lagascae_ ’s mitochondrial genome in SOAPdenovo. To annotate its mitochondrial genome, we used Mitofy (Alverson et al. 2010). Annotated ORFs with their accompanying numbers are based on the NCBI GenBank database. HGT of the mitochondrial genes was detected using phylogenetic analyses in MEGA5 (Tamura et al. 2011). In another approach, we did a de novo assembly of the _R. lagascae_ sequence reads using CLC Genomics Workbench (ver. 5.5.1) (CLC Bio, Aarhus, Denmark).
### Plastid Sequence Identification and Analysis
To identify sequence contigs comprising the plastid genome, we employed multiple approaches. First, sequence contigs from _R. lagascae_ identified by the CLC Genomics Workbench were mapped using Geneious R6 (Biomatters Ltd., Auckland, New Zealand). These contigs were first mapped to the full mitochondrial genomes of _V_ _. vinifera_ (GenBank accession number FM179380) and _Ri_ _. communis_ (HQ874649) to eliminate mitochondrial sequences, and all unmapped reads were then collected and mapped to the chloroplast genomes of _Vitis_ (DQ424856.1) and _Ricinus_ (JF937588.1). _Ricinus communis_ , like _R. lagascae_ , is in the Malpighiales, and is the closest species to _Rafflesia_ for which mitochondrial genome sequences are available. _Vitis vinifera_ is the closest species to _Tetrastigma_ for which both mitochondrial and chloroplast genome sequences are also available. Second, we conducted a BlastN search (e-value < e−10) of all the conserved plastid genes found in angiosperms available from GenBank against the CLC-assembled sequence contigs. We also generated profile HMM (Henderson et al. 1997) from alignments of conserved plastid genes using HMMER (<http://hmmer.janelia.org/>, last accessed February 8, 2014), which develops probabilistic models (profile HMMs) and can detect more remote similarities in sequence searches. We then mapped all these plastid sequence contigs identified (Geneious, BlastN, HMM) to the chloroplast genome of another species in the Malpighiales, _Hevea brasiliensis_ (GenBank accession number NC_015308), using Geneious R6 (Biomatters, Ltd.). In a third approach, we mapped the 100-bp PE reads using BWA (Burrows–Wheeler alignment) (Li and Durbin 2010) and SAMtools (Sequence Alignment/Map) (Li et al. 2009) to angiosperm plastid sequences from GenBank, and then the resulting mapped reads were de novo assembled using SOAPdenovo (Luo et al. 2012).
Putative plastid sequences identified earlier were then aligned with homologous regions from other taxa whose chloroplast genomes are available in GenBank using default parameters in the Multiple Alignment using Fast Fourier Transform (MAFFT) program (Katoh and Toh 2008) and visually checked (supplementary data S1, Supplementary Material online). The alignments were analyzed phylogenetically in MEGA5 (Tamura et al. 2011) using maximum likelihood with 100 bootstrap replicates and applying the best substitution model with the lowest Bayesian information criterion scores (supplementary data S2, Supplementary Material online).
To determine whether any of these putative plastid sequences are expressed, we also mapped the reads from the _R. cantleyi_ transcriptome library (accession SRA052224) (Xi et al. 2012) using Bowtie 1.0.0 (Langmead et al. 2009), TopHat2 (Kim et al. 2013), and Cufflinks (Trapnell et al. 2010) to the resulting contigs.
### Comparative Levels of Mitochondrial and Plastid Genomes in Algal and Flowering Plant Species
To compare our results in _Rafflesia_ with the relative levels of mitochondrial and chloroplast genome DNA in various species, we used whole genome resequencing data from three monocot species (_Phoenix dactylifera_ , _O_ _. sativa_ , and _O_ _. glaberrima_), one eudicot species (_A_ _. thaliana_), and one photosynthetic algae (_Chlamydomonas reinhardtii_). DNA from these species was isolated from single-cell cultures (for _C. reinhardtii_), leaf or shoot tissue (_P. dactylifera_ , _O. sativa_ , _O. glaberrima_ , and _A. thaliana_), or nonphotosynthetic root tissue (_O. glaberrima_ and _A. thaliana_). Libraries were constructed with 100-bp insert sizes using Illumina Standard DNA Library or Nextera kits (Illumina, San Diego, CA) and were sequenced as either 50-bp or 100-bp PE reads using Illumina Hiseq 2000 at NYU Center for Genomics and Systems Biology in New York or Abu Dhabi to obtain between 50 and 250 million reads per sample. For all these sequences, we mapped the reads using BWA (Li and Durbin 2010) and SAMtools (Sequence Alignment/Map) (Li et al. 2009), with the species genome sequence available in GenBank as a reference sequence.
##  Supplementary Material
Supplementary data S1 and S2, figure S1, and table S1 are available at _Molecular Biology and Evolution_ online (<http://www.mbe.oxfordjournals.org/>).
Supplementary Data
supp_31_4_793__index.html (1.3KB, html) 
##  Acknowledgments
The sequencing data have been deposited in the NCBI Sequence Read Archive (accession number SRX434531). The authors are grateful to Mary Ann Cajano, Salud Pangan, Filipinas Natividad, Karen Renzaglia, Steve Rounsley, Joseph Morin, Sandra Yap, Neda Barghi, Steven Sullivan, Jane Carlton, Arturo Lluisma, and Cynthia Saloma for their help and support in various aspects of the project. The authors are also grateful for the help provided by the Department of Environment and Natural Resources-Protected Areas and Wildlife Bureau (DENR-PAWB) of the Philippines, and thank Director Theresa Mundita S. Lim, Josefina de Leon, and Cecile Garcia for facilitating issuance of collecting permits. For assistance in collecting _R. lagascae_ , they thank Jovencio Payba, Provincial Environment and Natural Resources Office (PENRO)/DENR-Region 2, Apolinario U. Unarce, Sumper Aresta, Tabuc, Rolando Echanique and Alfredo Gabriel and The Cagayan Valley Partners in People Development (CAVAPPED) through its President/CEO Perla A. Visorro. For assistance in collecting _R. philippensis_ (Mt. Banahaw), the authors thank Jerry R. Mendua, Ananias M. Cahilo Sr., Romeo R. Diala, Angeles Coronado, Salud Pangan, and MaryAnn O. Cajano. This work was supported by a start-up grant from LIU and a Visiting Professorship grant from the University of the Philippines to J.M., and grants from the US National Science Foundation Plant Genome Research Program (IOS 1126971) and the NYU Abu Dhabi Institute to M.D.P. This article is dedicated to the memory of Leonard Co, a leading Philippine botanist and _Rafflesia_ investigator, who passed away while conducting fieldwork.
##  References
  1. Alverson AJ, Wei X, Rice DW, Stern DB, Barry K, Palmer JD. Insights into the evolution of mitochondrial genome size from complete sequences of Citrullus lanatus and Cucurbita pepo (Cucurbitaceae) Mol Biol Evol. 2010;27:1436–1448. doi: 10.1093/molbev/msq029. [DOI] [PMC free article] [PubMed] [Google Scholar&author=AJ%20Alverson&author=X%20Wei&author=DW%20Rice&author=DB%20Stern&author=K%20Barry&volume=27&publication_year=2010&pages=1436-1448&pmid=20118192&doi=10.1093/molbev/msq029&)]
  2. Barbrook AC, Howe CJ, Purton S. Why are plastid genomes retained in non-photosynthetic organisms? Trends Plant Sci. 2006;11:101–108. doi: 10.1016/j.tplants.2005.12.004. [DOI] [PubMed] [Google Scholar]
  3. Barcelona JF, Pelser PB, Balete DS, Co LL. Taxonomy, ecology, and conservation status of Philippine Rafflesia (Rafflesiaceae) Blumea. 2009;54:77–93. [Google Scholar&author=JF%20Barcelona&author=PB%20Pelser&author=DS%20Balete&author=LL%20Co&volume=54&publication_year=2009&pages=77-93&)]
  4. Barkman TJ, Mcneal JR, Lim SH, Coat G, Croom HB, Young ND, dePamphilis CW. Mitochondrial DNA suggests at least 11 origins of parasitism in angiosperms and reveals genomic chimerism in parasitic plants. BMC Evol Biol. 2007;7:248. doi: 10.1186/1471-2148-7-248. [DOI] [PMC free article] [PubMed] [Google Scholar]
  5. Birschwilks M, Haupt S, Hofius D, Neumann S. Transfer of phloem-mobile substances from the host plants to the holoparasite Cuscuta sp. J Exp Bot. 2006;57:911–921. doi: 10.1093/jxb/erj076. [DOI] [PubMed] [Google Scholar]
  6. Blanchard JL, Schmidt GW. Pervasive migration of organellar DNA to the nucleus in plants. J Mol Evol. 1995;41:397–406. doi: 10.1007/BF00160310. [DOI] [PubMed] [Google Scholar]
  7. Bock R. Structure, function, and inheritance of plastid genomes. In: Bock R, editor. Cell and molecular biology of plastids: topics in current genetics. Berlin: Springer; 2007. pp. 29–63. [Google Scholar]
  8. Braukmann T, Kuzmina M, Stefanovic S. Plastid genome evolution across the genus Cuscuta (Convolvulaceae): two clades within subgenus Grammica exhibit extensive gene loss. J Exp Bot. 2013;64:977–989. doi: 10.1093/jxb/ers391. [DOI] [PMC free article] [PubMed] [Google Scholar:%20two%20clades%20within%20subgenus%20Grammica%20exhibit%20extensive%20gene%20loss&author=T%20Braukmann&author=M%20Kuzmina&author=S%20Stefanovic&volume=64&publication_year=2013&pages=977-989&pmid=23349139&doi=10.1093/jxb/ers391&)]
  9. Braukmann T, Stefanovic S. Plastid genome evolution in mycoheterotrophic Ericaceae. Plant Mol Biol. 2012;79:5–20. doi: 10.1007/s11103-012-9884-3. [DOI] [PubMed] [Google Scholar]
  10. Bromham L, Cowman PF, Lanfear R. Parasitic plants have increased rates of molecular evolution across all three genomes. BMC Evol Biol. 2013;13:126. doi: 10.1186/1471-2148-13-126. [DOI] [PMC free article] [PubMed] [Google Scholar]
  11. Bungard RA. Photosynthetic evolution in parasitic plants: insight from the chloroplast genome. BioEssays. 2004;26:235–247. doi: 10.1002/bies.10405. [DOI] [PubMed] [Google Scholar]
  12. Camara B, Hugueney P, Bouvier F, Kuntz M, Moneger R. Biochemistry and molecular biology of chromoplast development. Int Rev Cytol. 1995;163:175–247. doi: 10.1016/s0074-7696(08)62211-1. [DOI62211-1)] [PubMed] [Google Scholar62211-1&)]
  13. Colwell AE. [St Louis (WA)]: Washington University; 1994. Genome evolution in a nonphotosynthetic plant, Conopholis americana [PhD dissertation] [Google Scholar%5D:%20Washington%20University;%201994.%20Genome%20evolution%20in%20a%20nonphotosynthetic%20plant,%20Conopholis%20americana%20%5BPhD%20dissertation%5D)]
  14. Davis CC, Latvis M, Nickrent DL, Wurdack KJ, Baum DA. Floral gigantism in Rafflesiaceae. Science. 2007;315:1812. doi: 10.1126/science.1135260. [DOI] [PubMed] [Google Scholar]
  15. Davis C, Wurdack K. Host to parasite transfer in flowering plants: phylogenetic evidence from Malpighiales. Science. 2004;305:676–678. doi: 10.1126/science.1100671. [DOI] [PubMed] [Google Scholar]
  16. Goff LJ, Coleman AW. Fate of parasite and host organelle DNA during cellular transformation of red algae by their parasites. Plant Cell. 1995;7:1899–1911. doi: 10.1105/tpc.7.11.1899. [DOI] [PMC free article] [PubMed] [Google Scholar]
  17. Gould SB, Waller RF, McFadden GI. Plastid evolution. Annu Rev Plant Biol. 2008;59:491–517. doi: 10.1146/annurev.arplant.59.032607.092915. [DOI] [PubMed] [Google Scholar]
  18. Henderson J, Salzberg S, Fasman KH. Finding genes in DNA with a hidden Markov model. J Comput Biol. 1997;4:127–141. doi: 10.1089/cmb.1997.4.127. [DOI] [PubMed] [Google Scholar]
  19. Isono K, Niwa Y, Satoh K, Kobayashi H. Evidence for transcriptional regulation of plastid photosynthesis genes in Arabidopsis thaliana roots. Plant Physiol. 1997;114:623–630. doi: 10.1104/pp.114.2.623. [DOI] [PMC free article] [PubMed] [Google Scholar]
  20. Katoh K, Toh H. Recent developments in the MAFFT multiple sequence alignment program. Brief Bioinform. 2008;9:286–298. doi: 10.1093/bib/bbn013. [DOI] [PubMed] [Google Scholar]
  21. Keeling PJ. The endosymbiotic origin, diversification and fate of plastids. Philos Trans R Soc Lond B Biol Sci. 2010;365:729–748. doi: 10.1098/rstb.2009.0103. [DOI] [PMC free article] [PubMed] [Google Scholar]
  22. Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol. 2013;14:R36. doi: 10.1186/gb-2013-14-4-r36. [DOI] [PMC free article] [PubMed] [Google Scholar]
  23. Kleine T, Maier UG, Leister D. DNA transfer from organelles to the nucleus: the idiosyncratic genetics of endosymbiosis. Annu Rev Plant Biol. 2009;60:115–138. doi: 10.1146/annurev.arplant.043008.092119. [DOI] [PubMed] [Google Scholar]
  24. Krause K. From chloroplasts to “cryptic” plastids: evolution of plastid genomes in parasitic plants. Curr Genet. 2008;54:111–121. doi: 10.1007/s00294-008-0208-8. [DOI] [PubMed] [Google Scholar]
  25. Krause K. Piecing together the puzzle of parasitic plant plastome evolution. Planta. 2011;234:647–656. doi: 10.1007/s00425-011-1494-9. [DOI] [PubMed] [Google Scholar]
  26. Kress WJ, Erickson DL, Jones AF, Swenson NG, Perez R, Sanjur O, Bermingham E. Plant DNA barcodes and a community phylogeny of a tropical forest dynamics plot in Panama. Proc Natl Acad Sci U S A. 2009;106:18621–18626. doi: 10.1073/pnas.0909820106. [DOI] [PMC free article] [PubMed] [Google Scholar]
  27. Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 2009;10:R25. doi: 10.1186/gb-2009-10-3-r25. [DOI] [PMC free article] [PubMed] [Google Scholar]
  28. Li H, Durbin R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics. 2010;26:589–595. doi: 10.1093/bioinformatics/btp698. [DOI] [PMC free article] [PubMed] [Google Scholar]
  29. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup The sequence alignment/map (SAM) format and SAMtools. Bioinformatics. 2009;25:2078–2079. doi: 10.1093/bioinformatics/btp352. [DOI] [PMC free article] [PubMed] [Google Scholar%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-2079&pmid=19505943&doi=10.1093/bioinformatics/btp352&)]
  30. Li X, Zhang TC, Qiao Q, Ren Z, Zhao J, Yonezawa T, Hasegawa M, Crabbe MJ, Li J, Zhong Y. Complete chloroplast genome sequence of holoparasite Cistanche deserticola (Orobanchaceae) reveals gene loss and horizontal gene transfer from its host Haloxylon ammodendron (Chenopodiaceae) PLoS One. 2013;8:e58747. doi: 10.1371/journal.pone.0058747. [DOI] [PMC free article] [PubMed] [Google Scholar%20reveals%20gene%20loss%20and%20horizontal%20gene%20transfer%20from%20its%20host%20Haloxylon%20ammodendron%20\(Chenopodiaceae\)&author=X%20Li&author=TC%20Zhang&author=Q%20Qiao&author=Z%20Ren&author=J%20Zhao&volume=8&publication_year=2013&pages=e58747&pmid=23554920&doi=10.1371/journal.pone.0058747&)]
  31. Luo R, Liu B, Xie Y, Li Z, Huang W, Yuan J, He G, Chen Y, Pan Q, Liu Y, et al. SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler. GigaScience. 2012;1:18. doi: 10.1186/2047-217X-1-18. [DOI] [PMC free article] [PubMed] [Google Scholar]
  32. Maréchal E, Cesbron-Delauw MF. The apicoplast: a new member of the plastid family. Trends Plant Sci. 2001;6:200–205. doi: 10.1016/s1360-1385(01)01921-5. [DOI01921-5)] [PubMed] [Google Scholar01921-5&)]
  33. Martin W. Gene transfer from organelles to the nucleus: frequent and in big chunks. Proc Natl Acad Sci U S A. 2003;100:8612–8614. doi: 10.1073/pnas.1633606100. [DOI] [PMC free article] [PubMed] [Google Scholar]
  34. Martin W, Borst P. Secondary loss of chloroplast in trypanosomes. Proc Natl Acad Sci U S A. 2003;100:765–767. doi: 10.1073/pnas.0437776100. [DOI] [PMC free article] [PubMed] [Google Scholar]
  35. McFadden GI, Reith ME, Munholland J, Lang-Unnasch N. Plastid in human parasites. Nature. 1996;381:482. doi: 10.1038/381482a0. [DOI] [PubMed] [Google Scholar]
  36. Nais J. Borneo (Kota Kinabalu): Natural History Publications; 2001. Rafflesia of the world. [Google Scholar:%20Natural%20History%20Publications;%202001.%20Rafflesia%20of%20the%20world.)]
  37. Neuhaus HE, Emes MJ. Nonphotosynthetic metabolism in plastids. Annu Rev Plant Physiol Plant Mol Biol. 2000;51:111–140. doi: 10.1146/annurev.arplant.51.1.111. [DOI] [PubMed] [Google Scholar]
  38. Nickrent DL, Blarer A, Qiu Y-L, Vidal-Russell R, Anderson FE. Phylogenetic inference in Rafflesiales: the influence of rate heterogeneity and horizontal gene transfer. BMC Evol Biol. 2004;4:e40. doi: 10.1186/1471-2148-4-40. [DOI] [PMC free article] [PubMed] [Google Scholar]
  39. Nickrent DL, Duff J, Ouyang Y, dePamphilis CW. Do holoparasitic Santalales possess a plastid genome? Plant Mol Biol. 1997;34:717–729. doi: 10.1023/a:1005860632601. [DOI] [PubMed] [Google Scholar]
  40. Palmer JD. The mitochondrion that time forgot. Nature. 1997;387:454–455. doi: 10.1038/387454a0. [DOI] [PubMed] [Google Scholar]
  41. Pelser PB, Nickrent DL, Callado JRC, Barcelona JF. Mt. Banahaw reveals: the resurrection and neotypification of the name R. lagascae (Rafflesiaceae) and clues to the dispersal of Rafflesia seeds. Phytotaxa. 2013;131:35–40. [Google Scholar%20and%20clues%20to%20the%20dispersal%20of%20Rafflesia%20seeds&author=PB%20Pelser&author=DL%20Nickrent&author=JRC%20Callado&author=JF%20Barcelona&volume=131&publication_year=2013&pages=35-40&)]
  42. Rivarola M, Foster JT, Chan AP, Williams AL, Rice DW, Liu X, Melake-Berhan A, Creasy HH, Puiu D, Rosovitz MJ, et al. Castor bean organelle genome sequencing and worldwide genetic diversity analysis. PLoS ONE. 2011;6:e21743. doi: 10.1371/journal.pone.0021743. [DOI] [PMC free article] [PubMed] [Google Scholar]
  43. Roney J, Khatibi P, Westwood J. Cross-species translocation of mRNA from host plants into the parasitic plant dodder. Plant Physiol. 2007;143:1037–1043. doi: 10.1104/pp.106.088369. [DOI] [PMC free article] [PubMed] [Google Scholar]
  44. Sacchetti G, Romagnoli C, Nicoletti M, Di Fabio A, Bruni A, Poli F. Glandular trichomes of Calceolaria adscendens Lidl. (Scrophulariaceae): histochemistry, development and ultrastructure. Ann Bot. 1999;83:87–92. [Google Scholar:%20histochemistry,%20development%20and%20ultrastructure&author=G%20Sacchetti&author=C%20Romagnoli&author=M%20Nicoletti&author=A%20Di%20Fabio&author=A%20Bruni&volume=83&publication_year=1999&pages=87-92&)]
  45. Stegemann S, Keuthe M, Greiner S, Bock R. Horizontal transfer of chloroplast genomes between plant species. Proc Natl Acad Sci U S A. 2012;109:2434–2438. doi: 10.1073/pnas.1114076109. [DOI] [PMC free article] [PubMed] [Google Scholar]
  46. Talianova M, Janousek B. What can we learn from tobacco and other Solanaceae about horizontal DNA transfer? Am J Bot. 2011;98:1231–1242. doi: 10.3732/ajb.1000370. [DOI] [PubMed] [Google Scholar]
  47. Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S. MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. Mol Biol Evol. 2011;28:2731–2739. doi: 10.1093/molbev/msr121. [DOI] [PMC free article] [PubMed] [Google Scholar]
  48. Tanaka Y, Sasaki N, Ohmiya A. Biosynthesis of plant pigments: anthocyanins, betalains and carotenoids. Plant J. 2008;54:733–749. doi: 10.1111/j.1365-313X.2008.03447.x. [DOI] [PubMed] [Google Scholar]
  49. Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, van Baren MJ, Salzberg SL, Wold BJ, Pachter L. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28:511–515. doi: 10.1038/nbt.1621. [DOI] [PMC free article] [PubMed] [Google Scholar]
  50. Van der Giezen M, Tovar J, Clark CG. Mitochondrion-derived organelles in protists and fungi. Int Rev Cytol. 2005;244:175–225. doi: 10.1016/S0074-7696(05)44005-X. [DOI44005-X)] [PubMed] [Google Scholar44005-X&)]
  51. Wen-Zhe L, Hong-Fei LB, Zheng-Hai H. Ultrastructure of the multicellular nodules in Hypericum perforatum leaves. Acta Bot Sin. 2002;44:649–656. [Google Scholar]
  52. Westwood JH, Yoder JI, Timko MP, dePamphilis CW. The evolution of parasitism in plants. Trends Plant Sci. 2010;15:227–235. doi: 10.1016/j.tplants.2010.01.004. [DOI] [PubMed] [Google Scholar]
  53. Wicke S, Schneeweiss GM, dePamphilis CW, Müller KF, Quandt D. The evolution of the plastid chromosome in land plants: gene content, gene order, gene function. Plant Mol Biol. 2011;76:273–297. doi: 10.1007/s11103-011-9762-4. [DOI] [PMC free article] [PubMed] [Google Scholar]
  54. Wicke S, Müller KF, dePamphilis CW, Quandt D, Wickett NJ, Zhang Y, Renner SS, Schneeweiss GM. Mechanisms of functional and physical genome reduction in photosynthetic and nonphotosynthetic parasitic plants of the broomrape family. Plant Cell. 2013;25:3711–3725. doi: 10.1105/tpc.113.113373. [DOI] [PMC free article] [PubMed] [Google Scholar]
  55. Wise RR. The diversity of plastid form and function. In: Wise RR, Hoober JK, editors. The structure and function of plastids. Vol. 23. Dordrecht (The Netherlands): Springer; 2006. pp. 3–26. [Google Scholar]
  56. Wolfe KH, Morden CW, Palmer JD. Function and evolution of a minimal plastid genome from a nonphotosynthetic parasitic plant. Proc Natl Acad Sci U S A. 1992;89:10648–10652. doi: 10.1073/pnas.89.22.10648. [DOI] [PMC free article] [PubMed] [Google Scholar]
  57. Xi Z, Bradley RK, Wurdack KJ, Wong K, Sugumaran M, Bomblies K, Davis CC. Horizontal transfer of expressed genes in a parasitic flowering plant. BMC Genomics. 2012;13:227. doi: 10.1186/1471-2164-13-227. [DOI] [PMC free article] [PubMed] [Google Scholar]
  58. Xi Z, Wang Y, Bradley RK, Sugumaran M, Marx CJ, Rest JS, Davis CC. Massive mitochondrial gene transfer in a parasitic flowering plant clade. PLoS Genet. 2013;9:e1003265. doi: 10.1371/journal.pgen.1003265. [DOI] [PMC free article] [PubMed] [Google Scholar]


##  Associated Data
_This section collects any data citations, data availability statements, or supplementary materials included in this article._
### Supplementary Materials
Supplementary Data
supp_31_4_793__index.html (1.3KB, html) 
supp_msu051_Supplementary_Data_2.pdf (1.4MB, pdf) 
supp_msu051_Supplementary_Data_1.txt (159.8KB, txt) 
supp_msu051_Supplementary_Figure1.pptx (2.4MB, pptx) 
supp_msu051_Supplementary_Table1.xlsx (42.4KB, xlsx) 
* * *
Articles from Molecular Biology and Evolution are provided here courtesy of **Oxford University Press**

## ACTIONS
  *  View on publisher site 
  *  PDF (938.5 KB) 
  * Cite
  * Collections
  * Permalink
## PERMALINK
Copy


## RESOURCES
###  Similar articles 
###  Cited by other articles 
###  Links to NCBI Databases 
## On this page
  * Abstract
  * Introduction
  * Results
  * Discussion
  * Materials and Methods
  * Supplementary Material
  * Acknowledgments
  * References
  * Associated Data


## Cite
  * Copy
  *  Download .nbib .nbib 
  * Format: AMA  APA  MLA  NLM 


## Add to Collections
Create a new collection
Add to an existing collection
Name your collection *
Choose a collection 
Unable to load your collection due to an error  
Please try again
Add  Cancel 
Follow NCBI 
 NCBI on X (formerly known as Twitter)   NCBI on Facebook   NCBI on LinkedIn   NCBI on GitHub   NCBI RSS feed 
Connect with NLM 
 NLM on X (formerly known as Twitter)   NLM on Facebook   NLM on YouTube 
National Library of Medicine   
8600 Rockville Pike  
Bethesda, MD 20894
  *  Web Policies 
  *  FOIA 
  *  HHS Vulnerability Disclosure 


  *  Help 
  *  Accessibility 
  *  Careers 


  *  NLM 
  *  NIH 
  *  HHS 
  *  USA.gov 


Back to Top 
  *[*]: required